<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.5.56">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="S.P.L.M., Gustavo">
<meta name="dcterms.date" content="2025-05-21">

<title>Main outcomes</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for syntax highlighting */
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { display: inline-block; text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
</style>


<script src="Results_main_outcomes_files/libs/clipboard/clipboard.min.js"></script>
<script src="Results_main_outcomes_files/libs/quarto-html/quarto.js"></script>
<script src="Results_main_outcomes_files/libs/quarto-html/popper.min.js"></script>
<script src="Results_main_outcomes_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="Results_main_outcomes_files/libs/quarto-html/anchor.min.js"></script>
<link href="Results_main_outcomes_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="Results_main_outcomes_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="Results_main_outcomes_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="Results_main_outcomes_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="Results_main_outcomes_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">

  <script src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6"></script>
  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

<script type="text/javascript">
const typesetMath = (el) => {
  if (window.MathJax) {
    // MathJax Typeset
    window.MathJax.typeset([el]);
  } else if (window.katex) {
    // KaTeX Render
    var mathElements = el.getElementsByClassName("math");
    var macros = [];
    for (var i = 0; i < mathElements.length; i++) {
      var texText = mathElements[i].firstChild;
      if (mathElements[i].tagName == "SPAN") {
        window.katex.render(texText.data, mathElements[i], {
          displayMode: mathElements[i].classList.contains('display'),
          throwOnError: false,
          macros: macros,
          fleqn: false
        });
      }
    }
  }
}
window.Quarto = {
  typesetMath
};
</script>

<link rel="stylesheet" href="styles.css">
</head>

<body>

<div id="quarto-content" class="page-columns page-rows-contents page-layout-full">
<div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
  <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#getting-started" id="toc-getting-started" class="nav-link active" data-scroll-target="#getting-started">Getting started</a>
  <ul>
  <li><a href="#load-data" id="toc-load-data" class="nav-link" data-scroll-target="#load-data">Load data</a></li>
  <li><a href="#variables-of-interest" id="toc-variables-of-interest" class="nav-link" data-scroll-target="#variables-of-interest">Variables of interest</a></li>
  <li><a href="#select-variables" id="toc-select-variables" class="nav-link" data-scroll-target="#select-variables">Select variables</a></li>
  <li><a href="#filter-codebook" id="toc-filter-codebook" class="nav-link" data-scroll-target="#filter-codebook">Filter codebook</a></li>
  <li><a href="#scaling" id="toc-scaling" class="nav-link" data-scroll-target="#scaling">Scaling</a></li>
  </ul></li>
  <li><a href="#phase-angle" id="toc-phase-angle" class="nav-link" data-scroll-target="#phase-angle">Phase angle</a>
  <ul>
  <li><a href="#modelo-1" id="toc-modelo-1" class="nav-link" data-scroll-target="#modelo-1">Modelo 1</a></li>
  <li><a href="#modelo-2" id="toc-modelo-2" class="nav-link" data-scroll-target="#modelo-2">Modelo 2</a></li>
  <li><a href="#modelo-3" id="toc-modelo-3" class="nav-link" data-scroll-target="#modelo-3">Modelo 3</a></li>
  </ul></li>
  <li><a href="#pcr" id="toc-pcr" class="nav-link" data-scroll-target="#pcr">PCR</a>
  <ul>
  <li><a href="#scaling-1" id="toc-scaling-1" class="nav-link" data-scroll-target="#scaling-1">Scaling</a></li>
  <li><a href="#per-protocol" id="toc-per-protocol" class="nav-link" data-scroll-target="#per-protocol">Per protocol</a>
  <ul class="collapse">
  <li><a href="#heteroscedasticity" id="toc-heteroscedasticity" class="nav-link" data-scroll-target="#heteroscedasticity">Heteroscedasticity</a>
  <ul class="collapse">
  <li><a href="#glmm" id="toc-glmm" class="nav-link" data-scroll-target="#glmm">GLMM</a></li>
  <li><a href="#tweedie" id="toc-tweedie" class="nav-link" data-scroll-target="#tweedie">Tweedie</a></li>
  <li><a href="#other-models" id="toc-other-models" class="nav-link" data-scroll-target="#other-models">Other models</a></li>
  </ul></li>
  </ul></li>
  <li><a href="#modeling-compliance" id="toc-modeling-compliance" class="nav-link" data-scroll-target="#modeling-compliance">Modeling compliance</a>
  <ul class="collapse">
  <li><a href="#ideal-condition" id="toc-ideal-condition" class="nav-link" data-scroll-target="#ideal-condition">Ideal condition</a></li>
  <li><a href="#restrict-dataset" id="toc-restrict-dataset" class="nav-link" data-scroll-target="#restrict-dataset">Restrict dataset</a></li>
  <li><a href="#iv" id="toc-iv" class="nav-link" data-scroll-target="#iv">IV</a></li>
  <li><a href="#ipw" id="toc-ipw" class="nav-link" data-scroll-target="#ipw">IPW</a></li>
  </ul></li>
  </ul></li>
  <li><a href="#pam" id="toc-pam" class="nav-link" data-scroll-target="#pam">PAM</a></li>
  </ul>
<div class="quarto-alternate-formats"><h2>Other Formats</h2><ul><li><a href="Results_main_outcomes.pdf"><i class="bi bi-file-pdf"></i>PDF</a></li></ul></div></nav>
</div>
<main class="content column-page-left" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Main outcomes</h1>
</div>



<div class="quarto-title-meta column-page-left">

    <div>
    <div class="quarto-title-meta-heading">Author</div>
    <div class="quarto-title-meta-contents">
             <p>S.P.L.M., Gustavo </p>
          </div>
  </div>
    
    <div>
    <div class="quarto-title-meta-heading">Published</div>
    <div class="quarto-title-meta-contents">
      <p class="date">May 21, 2025</p>
    </div>
  </div>
  
    
  </div>
  


</header>


<section id="getting-started" class="level1">
<h1>Getting started</h1>
<section id="load-data" class="level2">
<h2 class="anchored" data-anchor-id="load-data">Load data</h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb1"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="fu">rm</span>(<span class="at">list =</span> <span class="fu">ls</span>())</span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a><span class="fu">graphics.off</span>()</span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a><span class="fu">cat</span>(<span class="st">"</span><span class="sc">\014</span><span class="st">"</span>)  <span class="co"># Clear any pending RStudio sessions or temporary files</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div style="page-break-after: always;"></div>
<div class="sourceCode cell-code" id="cb2"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb2-1"><a href="#cb2-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Load functions from external script</span></span>
<span id="cb2-2"><a href="#cb2-2" aria-hidden="true" tabindex="-1"></a><span class="fu">source</span>(<span class="st">"helper_functions.R"</span>)</span>
<span id="cb2-3"><a href="#cb2-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-4"><a href="#cb2-4" aria-hidden="true" tabindex="-1"></a><span class="do">## Load necessary libraries</span></span>
<span id="cb2-5"><a href="#cb2-5" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(tidyverse)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>── Attaching core tidyverse packages ──────────────────────── tidyverse 2.0.0 ──
✔ dplyr     1.1.4.9000     ✔ readr     2.1.5     
✔ forcats   1.0.0          ✔ stringr   1.5.1     
✔ ggplot2   3.5.1          ✔ tibble    3.2.1     
✔ lubridate 1.9.4          ✔ tidyr     1.3.1     
✔ purrr     1.0.4          
── Conflicts ────────────────────────────────────────── tidyverse_conflicts() ──
✖ dplyr::filter() masks stats::filter()
✖ dplyr::lag()    masks stats::lag()
ℹ Use the conflicted package (&lt;http://conflicted.r-lib.org/&gt;) to force all conflicts to become errors</code></pre>
</div>
<div class="sourceCode cell-code" id="cb4"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb4-1"><a href="#cb4-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(readxl)</span>
<span id="cb4-2"><a href="#cb4-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(lubridate)</span>
<span id="cb4-3"><a href="#cb4-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(stringr)</span>
<span id="cb4-4"><a href="#cb4-4" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(purrr)</span>
<span id="cb4-5"><a href="#cb4-5" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(here)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>here() starts at /Users/gustavosplmoura/Library/Mobile Documents/com~apple~CloudDocs/Medicina/Biblioteca/Data Science/Data Science/R_Projects_Research/DVEP</code></pre>
</div>
<div class="sourceCode cell-code" id="cb6"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb6-1"><a href="#cb6-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(lme4)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Loading required package: Matrix

Attaching package: 'Matrix'

The following objects are masked from 'package:tidyr':

    expand, pack, unpack</code></pre>
</div>
<div class="sourceCode cell-code" id="cb8"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb8-1"><a href="#cb8-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(lmerTest)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>
Attaching package: 'lmerTest'

The following object is masked from 'package:lme4':

    lmer

The following object is masked from 'package:stats':

    step</code></pre>
</div>
<div class="sourceCode cell-code" id="cb10"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb10-1"><a href="#cb10-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(skimr)</span>
<span id="cb10-2"><a href="#cb10-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(performance)</span>
<span id="cb10-3"><a href="#cb10-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb10-4"><a href="#cb10-4" aria-hidden="true" tabindex="-1"></a><span class="co"># Read Files ----</span></span>
<span id="cb10-5"><a href="#cb10-5" aria-hidden="true" tabindex="-1"></a><span class="do">## Codebooks</span></span>
<span id="cb10-6"><a href="#cb10-6" aria-hidden="true" tabindex="-1"></a>codebook_dvep <span class="ot">&lt;-</span> <span class="fu">read_excel</span>(</span>
<span id="cb10-7"><a href="#cb10-7" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Codebooks/codebook_dvep.xlsx"</span>,</span>
<span id="cb10-8"><a href="#cb10-8" aria-hidden="true" tabindex="-1"></a>    <span class="at">col_names =</span> <span class="cn">TRUE</span>,</span>
<span id="cb10-9"><a href="#cb10-9" aria-hidden="true" tabindex="-1"></a>    <span class="at">col_types =</span> <span class="cn">NULL</span>,</span>
<span id="cb10-10"><a href="#cb10-10" aria-hidden="true" tabindex="-1"></a>    <span class="at">na =</span> <span class="fu">c</span>(<span class="st">""</span>, <span class="st">"NA"</span>, <span class="st">"NI"</span>, <span class="st">"UNK"</span>, <span class="st">"NASK"</span>, <span class="st">"ASKU"</span>, <span class="st">"INV"</span>),</span>
<span id="cb10-11"><a href="#cb10-11" aria-hidden="true" tabindex="-1"></a>    <span class="at">trim_ws =</span> <span class="cn">TRUE</span>,</span>
<span id="cb10-12"><a href="#cb10-12" aria-hidden="true" tabindex="-1"></a>    <span class="at">skip =</span> <span class="dv">0</span>, <span class="co"># Number of lines to skip before reading data</span></span>
<span id="cb10-13"><a href="#cb10-13" aria-hidden="true" tabindex="-1"></a>    <span class="at">n_max =</span> <span class="cn">Inf</span>, <span class="co"># Maximum number of lines to read.</span></span>
<span id="cb10-14"><a href="#cb10-14" aria-hidden="true" tabindex="-1"></a>    <span class="at">guess_max =</span> <span class="dv">1000</span></span>
<span id="cb10-15"><a href="#cb10-15" aria-hidden="true" tabindex="-1"></a>) <span class="sc">%&gt;%</span></span>
<span id="cb10-16"><a href="#cb10-16" aria-hidden="true" tabindex="-1"></a>    <span class="fu">arrange</span>(index)</span>
<span id="cb10-17"><a href="#cb10-17" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb10-18"><a href="#cb10-18" aria-hidden="true" tabindex="-1"></a>codebook_structure  <span class="ot">&lt;-</span> <span class="fu">read_csv</span>(</span>
<span id="cb10-19"><a href="#cb10-19" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Codebooks/codebook_structure.csv"</span>,</span>
<span id="cb10-20"><a href="#cb10-20" aria-hidden="true" tabindex="-1"></a>    <span class="at">col_names =</span> <span class="cn">TRUE</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Rows: 34 Columns: 9
── Column specification ────────────────────────────────────────────────────────
Delimiter: ","
chr (2): form_name_en, form_name_pt
dbl (7): repeating, eleg, V1, V2, V3, order, variable_count

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.</code></pre>
</div>
<div class="sourceCode cell-code" id="cb12"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb12-1"><a href="#cb12-1" aria-hidden="true" tabindex="-1"></a>codebook_ncit  <span class="ot">&lt;-</span> <span class="fu">read_csv</span>(</span>
<span id="cb12-2"><a href="#cb12-2" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Codebooks/codebook_ncit.csv"</span>,</span>
<span id="cb12-3"><a href="#cb12-3" aria-hidden="true" tabindex="-1"></a>    <span class="at">col_names =</span> <span class="cn">TRUE</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Rows: 330 Columns: 4
── Column specification ────────────────────────────────────────────────────────
Delimiter: ","
chr (2): ncit_code, descriptive
dbl (2): type, medicamentos_comorbidades_complete

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.</code></pre>
</div>
<div class="sourceCode cell-code" id="cb14"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb14-1"><a href="#cb14-1" aria-hidden="true" tabindex="-1"></a>codebook_bia <span class="ot">&lt;-</span> <span class="fu">read_excel</span>(</span>
<span id="cb14-2"><a href="#cb14-2" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Codebooks/codebook_bia.xlsx"</span>,</span>
<span id="cb14-3"><a href="#cb14-3" aria-hidden="true" tabindex="-1"></a>    <span class="at">col_names =</span> <span class="cn">TRUE</span>,</span>
<span id="cb14-4"><a href="#cb14-4" aria-hidden="true" tabindex="-1"></a>    <span class="at">col_types =</span> <span class="cn">NULL</span>,</span>
<span id="cb14-5"><a href="#cb14-5" aria-hidden="true" tabindex="-1"></a>    <span class="at">na =</span> <span class="fu">c</span>(<span class="st">""</span>, <span class="st">"NA"</span>, <span class="st">"NI"</span>, <span class="st">"UNK"</span>, <span class="st">"NASK"</span>, <span class="st">"ASKU"</span>, <span class="st">"INV"</span>),</span>
<span id="cb14-6"><a href="#cb14-6" aria-hidden="true" tabindex="-1"></a>    <span class="at">trim_ws =</span> <span class="cn">TRUE</span>,</span>
<span id="cb14-7"><a href="#cb14-7" aria-hidden="true" tabindex="-1"></a>    <span class="at">skip =</span> <span class="dv">0</span>, <span class="co"># Number of lines to skip before reading data</span></span>
<span id="cb14-8"><a href="#cb14-8" aria-hidden="true" tabindex="-1"></a>    <span class="at">n_max =</span> <span class="cn">Inf</span>, <span class="co"># Maximum number of lines to read.</span></span>
<span id="cb14-9"><a href="#cb14-9" aria-hidden="true" tabindex="-1"></a>    <span class="at">guess_max =</span> <span class="dv">1000</span></span>
<span id="cb14-10"><a href="#cb14-10" aria-hidden="true" tabindex="-1"></a>) <span class="sc">%&gt;%</span></span>
<span id="cb14-11"><a href="#cb14-11" aria-hidden="true" tabindex="-1"></a>    <span class="fu">arrange</span>(index)</span>
<span id="cb14-12"><a href="#cb14-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb14-13"><a href="#cb14-13" aria-hidden="true" tabindex="-1"></a><span class="do">## Data</span></span>
<span id="cb14-14"><a href="#cb14-14" aria-hidden="true" tabindex="-1"></a>data <span class="ot">&lt;-</span> <span class="fu">readRDS</span>(<span class="st">"Data_processed/data.rds"</span>)</span>
<span id="cb14-15"><a href="#cb14-15" aria-hidden="true" tabindex="-1"></a>data_bia <span class="ot">&lt;-</span> <span class="fu">readRDS</span>(<span class="st">"Data_processed/data_bia.rds"</span>)</span>
<span id="cb14-16"><a href="#cb14-16" aria-hidden="true" tabindex="-1"></a>data_bia_D1 <span class="ot">&lt;-</span> <span class="fu">readRDS</span>(<span class="st">"Data_processed/data_bia_D1.rds"</span>)</span>
<span id="cb14-17"><a href="#cb14-17" aria-hidden="true" tabindex="-1"></a>data_bia_D1_mean <span class="ot">&lt;-</span> <span class="fu">readRDS</span>(<span class="st">"Data_processed/data_bia_D1_mean.rds"</span>)</span>
<span id="cb14-18"><a href="#cb14-18" aria-hidden="true" tabindex="-1"></a>data_bia_D1_raw <span class="ot">&lt;-</span> <span class="fu">readRDS</span>(<span class="st">"Data_processed/data_bia_D1_raw.rds"</span>)</span>
<span id="cb14-19"><a href="#cb14-19" aria-hidden="true" tabindex="-1"></a>data_bia_D3 <span class="ot">&lt;-</span> <span class="fu">readRDS</span>(<span class="st">"Data_processed/data_bia_D3.rds"</span>)</span>
<span id="cb14-20"><a href="#cb14-20" aria-hidden="true" tabindex="-1"></a>data_bia_D3_mean <span class="ot">&lt;-</span> <span class="fu">readRDS</span>(<span class="st">"Data_processed/data_bia_D3_mean.rds"</span>)</span>
<span id="cb14-21"><a href="#cb14-21" aria-hidden="true" tabindex="-1"></a>data_bia_D3_raw <span class="ot">&lt;-</span> <span class="fu">readRDS</span>(<span class="st">"Data_processed/data_bia_D3_raw.rds"</span>)</span>
<span id="cb14-22"><a href="#cb14-22" aria-hidden="true" tabindex="-1"></a>data_bia_mean <span class="ot">&lt;-</span> <span class="fu">readRDS</span>(<span class="st">"Data_processed/data_bia_mean.rds"</span>)</span>
<span id="cb14-23"><a href="#cb14-23" aria-hidden="true" tabindex="-1"></a>data_d1_exclusive <span class="ot">&lt;-</span> <span class="fu">readRDS</span>(<span class="st">"Data_processed/data_d1_exclusive.rds"</span>)</span>
<span id="cb14-24"><a href="#cb14-24" aria-hidden="true" tabindex="-1"></a>data_filtered <span class="ot">&lt;-</span> <span class="fu">readRDS</span>(<span class="st">"Data_processed/data_filtered.rds"</span>)</span>
<span id="cb14-25"><a href="#cb14-25" aria-hidden="true" tabindex="-1"></a>data_filtered_seca <span class="ot">&lt;-</span> <span class="fu">readRDS</span>(<span class="st">"Data_processed/data_filtered_seca.rds"</span>)</span>
<span id="cb14-26"><a href="#cb14-26" aria-hidden="true" tabindex="-1"></a>I21_conditions_R <span class="ot">&lt;-</span> <span class="fu">readRDS</span>(<span class="st">"Data_processed/I21_conditions_R.rds"</span>)</span>
<span id="cb14-27"><a href="#cb14-27" aria-hidden="true" tabindex="-1"></a>I22_drugs_R <span class="ot">&lt;-</span> <span class="fu">readRDS</span>(<span class="st">"Data_processed/I22_drugs_R.rds"</span>)</span>
<span id="cb14-28"><a href="#cb14-28" aria-hidden="true" tabindex="-1"></a>I27_labs_R <span class="ot">&lt;-</span> <span class="fu">readRDS</span>(<span class="st">"Data_processed/I27_labs_R.rds"</span>)</span>
<span id="cb14-29"><a href="#cb14-29" aria-hidden="true" tabindex="-1"></a>I29_compliance_new <span class="ot">&lt;-</span> <span class="fu">readRDS</span>(<span class="st">"Data_processed/I29_compliance_new.rds"</span>)</span>
<span id="cb14-30"><a href="#cb14-30" aria-hidden="true" tabindex="-1"></a>I30_events_R <span class="ot">&lt;-</span> <span class="fu">readRDS</span>(<span class="st">"Data_processed/I30_events_R.rds"</span>)</span>
<span id="cb14-31"><a href="#cb14-31" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb14-32"><a href="#cb14-32" aria-hidden="true" tabindex="-1"></a><span class="do">## SUPERTIBBLE</span></span>
<span id="cb14-33"><a href="#cb14-33" aria-hidden="true" tabindex="-1"></a>data_instruments <span class="ot">&lt;-</span> <span class="fu">readRDS</span>(<span class="st">"Data_instruments/data_instruments.rds"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="variables-of-interest" class="level2">
<h2 class="anchored" data-anchor-id="variables-of-interest">Variables of interest</h2>
<ul>
<li>(1 | record_id) + visit + allocation_group + age + sex + race + education +</li>
<li>Comorbidities: hypertension + hypercholesterolemia + hypertrigliceridemia + insulin + drugs_w_loss + drugs_w_gain</li>
<li>Anthro: abdomen + bmi + mean_bp_mean</li>
<li>Questionnaires: whoqol_score_overall + ecap_score + evs_score + dass_score_stress + dass_score_anxiety + dass_score_depression + alcohol_significant + smoke_history + carbs_kcal + protein_kcal + fat_kcal + drugs_dose_change_yn +</li>
<li>Adesão: completed_intervention, intervention_duration, education_years? cp_taking_as_directed_yn, cp_missed_dose_yn, cp_missed_dose_count, cp_discontinued_yn, cp_discontinued_n_days, cp_ran_out_of_drug_yn, cp_medication_confidence_sca</li>
</ul>
</section>
<section id="select-variables" class="level2">
<h2 class="anchored" data-anchor-id="select-variables">Select variables</h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb15"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb15-1"><a href="#cb15-1" aria-hidden="true" tabindex="-1"></a>data_model <span class="ot">&lt;-</span> data_filtered <span class="sc">%&gt;%</span> </span>
<span id="cb15-2"><a href="#cb15-2" aria-hidden="true" tabindex="-1"></a>    <span class="fu">select</span>(</span>
<span id="cb15-3"><a href="#cb15-3" aria-hidden="true" tabindex="-1"></a>        record_id<span class="sc">:</span>sex,</span>
<span id="cb15-4"><a href="#cb15-4" aria-hidden="true" tabindex="-1"></a>        hypertension<span class="sc">:</span>ecap_score,</span>
<span id="cb15-5"><a href="#cb15-5" aria-hidden="true" tabindex="-1"></a>        abdomen, bmi, mean_bp_mean,</span>
<span id="cb15-6"><a href="#cb15-6" aria-hidden="true" tabindex="-1"></a>        resistance<span class="sc">:</span>evs_score,</span>
<span id="cb15-7"><a href="#cb15-7" aria-hidden="true" tabindex="-1"></a>        alcohol_dose, </span>
<span id="cb15-8"><a href="#cb15-8" aria-hidden="true" tabindex="-1"></a>        carbs_kcal, protein_kcal, fat_kcal, kcal,</span>
<span id="cb15-9"><a href="#cb15-9" aria-hidden="true" tabindex="-1"></a>        labs_crp<span class="sc">:</span>labs_alkp,</span>
<span id="cb15-10"><a href="#cb15-10" aria-hidden="true" tabindex="-1"></a>        labs_cholesterol<span class="sc">:</span>labs_quick_index</span>
<span id="cb15-11"><a href="#cb15-11" aria-hidden="true" tabindex="-1"></a>    )</span>
<span id="cb15-12"><a href="#cb15-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb15-13"><a href="#cb15-13" aria-hidden="true" tabindex="-1"></a><span class="fu">saveRDS</span>(</span>
<span id="cb15-14"><a href="#cb15-14" aria-hidden="true" tabindex="-1"></a>    data_model,</span>
<span id="cb15-15"><a href="#cb15-15" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Data_processed/data_model.rds"</span>)</span>
<span id="cb15-16"><a href="#cb15-16" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb15-17"><a href="#cb15-17" aria-hidden="true" tabindex="-1"></a>vars_to_keep <span class="ot">&lt;-</span> <span class="fu">names</span>(data_model)</span>
<span id="cb15-18"><a href="#cb15-18" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb15-19"><a href="#cb15-19" aria-hidden="true" tabindex="-1"></a><span class="co"># Step 2: filter codebook_dvep</span></span>
<span id="cb15-20"><a href="#cb15-20" aria-hidden="true" tabindex="-1"></a>codebook_data_model <span class="ot">&lt;-</span> codebook_dvep <span class="sc">%&gt;%</span></span>
<span id="cb15-21"><a href="#cb15-21" aria-hidden="true" tabindex="-1"></a>    <span class="fu">filter</span>(variable <span class="sc">%in%</span> vars_to_keep) <span class="sc">%&gt;%</span> </span>
<span id="cb15-22"><a href="#cb15-22" aria-hidden="true" tabindex="-1"></a>    <span class="fu">select</span>(</span>
<span id="cb15-23"><a href="#cb15-23" aria-hidden="true" tabindex="-1"></a>        variable, label_pt, field_type, choices)</span>
<span id="cb15-24"><a href="#cb15-24" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb15-25"><a href="#cb15-25" aria-hidden="true" tabindex="-1"></a><span class="fu">saveRDS</span>(</span>
<span id="cb15-26"><a href="#cb15-26" aria-hidden="true" tabindex="-1"></a>    codebook_data_model,</span>
<span id="cb15-27"><a href="#cb15-27" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Data_processed/codebook_data_model.rds"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="filter-codebook" class="level2">
<h2 class="anchored" data-anchor-id="filter-codebook">Filter codebook</h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb16"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb16-1"><a href="#cb16-1" aria-hidden="true" tabindex="-1"></a>codebook_filtered <span class="ot">&lt;-</span> codebook_dvep <span class="sc">%&gt;%</span></span>
<span id="cb16-2"><a href="#cb16-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(variable <span class="sc">%in%</span> <span class="fu">colnames</span>(data_model))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="scaling" class="level2">
<h2 class="anchored" data-anchor-id="scaling">Scaling</h2>
<p><span class="math display">\[{QUICKI} = \frac{1}{\\log(insulin) + log(glucose)}\]</span></p>
<p><span class="math display">\[
HOMA-IR = \frac{insulin * glucose}{405}
\]</span></p>
<div class="cell">
<div class="sourceCode cell-code" id="cb17"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb17-1"><a href="#cb17-1" aria-hidden="true" tabindex="-1"></a>data_model_scaled <span class="ot">&lt;-</span> data_model <span class="sc">%&gt;%</span></span>
<span id="cb17-2"><a href="#cb17-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="fu">across</span>(</span>
<span id="cb17-3"><a href="#cb17-3" aria-hidden="true" tabindex="-1"></a>    <span class="at">.cols =</span> <span class="fu">c</span>(</span>
<span id="cb17-4"><a href="#cb17-4" aria-hidden="true" tabindex="-1"></a>        duration_difference, age,</span>
<span id="cb17-5"><a href="#cb17-5" aria-hidden="true" tabindex="-1"></a>        whoqol_score_overall, dass_score_depression, dass_score_anxiety,</span>
<span id="cb17-6"><a href="#cb17-6" aria-hidden="true" tabindex="-1"></a>        dass_score_stress, ecap_score, </span>
<span id="cb17-7"><a href="#cb17-7" aria-hidden="true" tabindex="-1"></a>        abdomen, bmi, mean_bp_mean,</span>
<span id="cb17-8"><a href="#cb17-8" aria-hidden="true" tabindex="-1"></a>        resistance, reactance, </span>
<span id="cb17-9"><a href="#cb17-9" aria-hidden="true" tabindex="-1"></a>        handgrip, evs_score, alcohol_dose,</span>
<span id="cb17-10"><a href="#cb17-10" aria-hidden="true" tabindex="-1"></a>        carbs_kcal, protein_kcal, fat_kcal, kcal,</span>
<span id="cb17-11"><a href="#cb17-11" aria-hidden="true" tabindex="-1"></a>        labs_crp, labs_ast, labs_alt, labs_ggt, labs_alkp,</span>
<span id="cb17-12"><a href="#cb17-12" aria-hidden="true" tabindex="-1"></a>        labs_cholesterol, labs_ldl, labs_hba1c, labs_triglycerides,</span>
<span id="cb17-13"><a href="#cb17-13" aria-hidden="true" tabindex="-1"></a>        labs_hdl, labs_glucose, labs_insulin, labs_homa_ir</span>
<span id="cb17-14"><a href="#cb17-14" aria-hidden="true" tabindex="-1"></a>    ),</span>
<span id="cb17-15"><a href="#cb17-15" aria-hidden="true" tabindex="-1"></a>    <span class="at">.fns =</span> <span class="sc">~</span> <span class="fu">as.numeric</span>(<span class="fu">scale</span>(.))</span>
<span id="cb17-16"><a href="#cb17-16" aria-hidden="true" tabindex="-1"></a>  )) </span>
<span id="cb17-17"><a href="#cb17-17" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb17-18"><a href="#cb17-18" aria-hidden="true" tabindex="-1"></a>scaling_params <span class="ot">&lt;-</span> data_model <span class="sc">%&gt;%</span></span>
<span id="cb17-19"><a href="#cb17-19" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarise</span>(<span class="fu">across</span>(</span>
<span id="cb17-20"><a href="#cb17-20" aria-hidden="true" tabindex="-1"></a>    <span class="at">.cols =</span> <span class="fu">c</span>(</span>
<span id="cb17-21"><a href="#cb17-21" aria-hidden="true" tabindex="-1"></a>      duration_difference, age,</span>
<span id="cb17-22"><a href="#cb17-22" aria-hidden="true" tabindex="-1"></a>      whoqol_score_overall, dass_score_depression, dass_score_anxiety,</span>
<span id="cb17-23"><a href="#cb17-23" aria-hidden="true" tabindex="-1"></a>      dass_score_stress, ecap_score, </span>
<span id="cb17-24"><a href="#cb17-24" aria-hidden="true" tabindex="-1"></a>      abdomen, bmi, mean_bp_mean,</span>
<span id="cb17-25"><a href="#cb17-25" aria-hidden="true" tabindex="-1"></a>      resistance, reactance, </span>
<span id="cb17-26"><a href="#cb17-26" aria-hidden="true" tabindex="-1"></a>      handgrip, evs_score, alcohol_dose,</span>
<span id="cb17-27"><a href="#cb17-27" aria-hidden="true" tabindex="-1"></a>      carbs_kcal, protein_kcal, fat_kcal,</span>
<span id="cb17-28"><a href="#cb17-28" aria-hidden="true" tabindex="-1"></a>      labs_crp, labs_ast, labs_alt, labs_ggt, labs_alkp,</span>
<span id="cb17-29"><a href="#cb17-29" aria-hidden="true" tabindex="-1"></a>      labs_cholesterol, labs_ldl, labs_hba1c, labs_triglycerides,</span>
<span id="cb17-30"><a href="#cb17-30" aria-hidden="true" tabindex="-1"></a>      labs_hdl, labs_glucose, labs_insulin, labs_homa_ir</span>
<span id="cb17-31"><a href="#cb17-31" aria-hidden="true" tabindex="-1"></a>    ),</span>
<span id="cb17-32"><a href="#cb17-32" aria-hidden="true" tabindex="-1"></a>    <span class="fu">list</span>(<span class="at">mean =</span> <span class="sc">~</span><span class="fu">mean</span>(., <span class="at">na.rm =</span> <span class="cn">TRUE</span>), <span class="at">sd =</span> <span class="sc">~</span><span class="fu">sd</span>(., <span class="at">na.rm =</span> <span class="cn">TRUE</span>))</span>
<span id="cb17-33"><a href="#cb17-33" aria-hidden="true" tabindex="-1"></a>  )) <span class="sc">%&gt;%</span></span>
<span id="cb17-34"><a href="#cb17-34" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pivot_longer</span>(<span class="fu">everything</span>(),</span>
<span id="cb17-35"><a href="#cb17-35" aria-hidden="true" tabindex="-1"></a>               <span class="at">names_to =</span> <span class="fu">c</span>(<span class="st">"variable"</span>, <span class="st">".value"</span>),</span>
<span id="cb17-36"><a href="#cb17-36" aria-hidden="true" tabindex="-1"></a>               <span class="at">names_sep =</span> <span class="st">"_(?=[^_]+$)"</span>,  <span class="co"># 👈 this fixes the splitting</span></span>
<span id="cb17-37"><a href="#cb17-37" aria-hidden="true" tabindex="-1"></a>               <span class="at">names_transform =</span> <span class="fu">list</span>(<span class="at">.value =</span> as.character))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb18"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb18-1"><a href="#cb18-1" aria-hidden="true" tabindex="-1"></a><span class="co"># THEN, FOR NEW DATA</span></span>
<span id="cb18-2"><a href="#cb18-2" aria-hidden="true" tabindex="-1"></a>scale_with_params <span class="ot">&lt;-</span> <span class="cf">function</span>(new_data, params) {</span>
<span id="cb18-3"><a href="#cb18-3" aria-hidden="true" tabindex="-1"></a>  <span class="cf">for</span> (i <span class="cf">in</span> <span class="fu">seq_len</span>(<span class="fu">nrow</span>(params))) {</span>
<span id="cb18-4"><a href="#cb18-4" aria-hidden="true" tabindex="-1"></a>    var <span class="ot">&lt;-</span> params<span class="sc">$</span>variable[i]</span>
<span id="cb18-5"><a href="#cb18-5" aria-hidden="true" tabindex="-1"></a>    mean_val <span class="ot">&lt;-</span> params<span class="sc">$</span>mean[i]</span>
<span id="cb18-6"><a href="#cb18-6" aria-hidden="true" tabindex="-1"></a>    sd_val <span class="ot">&lt;-</span> params<span class="sc">$</span>sd[i]</span>
<span id="cb18-7"><a href="#cb18-7" aria-hidden="true" tabindex="-1"></a>    <span class="cf">if</span> (var <span class="sc">%in%</span> <span class="fu">names</span>(new_data)) {</span>
<span id="cb18-8"><a href="#cb18-8" aria-hidden="true" tabindex="-1"></a>      new_data[[var]] <span class="ot">&lt;-</span> (new_data[[var]] <span class="sc">-</span> mean_val) <span class="sc">/</span> sd_val</span>
<span id="cb18-9"><a href="#cb18-9" aria-hidden="true" tabindex="-1"></a>    }</span>
<span id="cb18-10"><a href="#cb18-10" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb18-11"><a href="#cb18-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">return</span>(new_data)</span>
<span id="cb18-12"><a href="#cb18-12" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb18-13"><a href="#cb18-13" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb18-14"><a href="#cb18-14" aria-hidden="true" tabindex="-1"></a>new_data_scaled <span class="ot">&lt;-</span> <span class="fu">scale_with_params</span>(new_data, scaling_params)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
</section>
<section id="phase-angle" class="level1">
<h1>Phase angle</h1>
<p>Filtrando os dados da primeira e terceira visitas.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb19"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb19-1"><a href="#cb19-1" aria-hidden="true" tabindex="-1"></a>pha_redcap <span class="ot">&lt;-</span> data_model_scaled <span class="sc">%&gt;%</span> </span>
<span id="cb19-2"><a href="#cb19-2" aria-hidden="true" tabindex="-1"></a>    <span class="fu">filter</span>(</span>
<span id="cb19-3"><a href="#cb19-3" aria-hidden="true" tabindex="-1"></a>        visit <span class="sc">%in%</span> <span class="fu">c</span>(<span class="dv">1</span>, <span class="dv">3</span>)</span>
<span id="cb19-4"><a href="#cb19-4" aria-hidden="true" tabindex="-1"></a>    ) <span class="sc">%&gt;%</span> </span>
<span id="cb19-5"><a href="#cb19-5" aria-hidden="true" tabindex="-1"></a>    <span class="fu">mutate</span>(</span>
<span id="cb19-6"><a href="#cb19-6" aria-hidden="true" tabindex="-1"></a>        <span class="at">compliance_score_visit =</span> <span class="fu">case_when</span>(</span>
<span id="cb19-7"><a href="#cb19-7" aria-hidden="true" tabindex="-1"></a>            visit <span class="sc">==</span> <span class="dv">3</span> <span class="sc">&amp;</span> completed_intervention <span class="sc">==</span> <span class="st">"Não"</span> <span class="sc">~</span> <span class="dv">0</span>,</span>
<span id="cb19-8"><a href="#cb19-8" aria-hidden="true" tabindex="-1"></a>            <span class="cn">TRUE</span> <span class="sc">~</span> compliance_score_visit</span>
<span id="cb19-9"><a href="#cb19-9" aria-hidden="true" tabindex="-1"></a>        )</span>
<span id="cb19-10"><a href="#cb19-10" aria-hidden="true" tabindex="-1"></a>    )</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<section id="modelo-1" class="level2">
<h2 class="anchored" data-anchor-id="modelo-1">Modelo 1</h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb20"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb20-1"><a href="#cb20-1" aria-hidden="true" tabindex="-1"></a>pha_1 <span class="ot">&lt;-</span> <span class="fu">lmer</span>(phase_angle <span class="sc">~</span> (<span class="dv">1</span> <span class="sc">|</span> record_id) <span class="sc">+</span> visit <span class="sc">+</span> allocation_group <span class="sc">+</span> completed_intervention <span class="sc">+</span> duration_difference <span class="sc">+</span> age <span class="sc">+</span> sex <span class="sc">+</span> hypertension <span class="sc">+</span> hypercholesterolemia <span class="sc">+</span> hypertrigliceridemia <span class="sc">+</span> drugs_w_loss <span class="sc">+</span> drugs_w_gain <span class="sc">+</span> mean_bp_mean <span class="sc">+</span> handgrip <span class="sc">+</span> evs_score <span class="sc">+</span> alcohol_dose <span class="sc">+</span> kcal <span class="sc">+</span> labs_crp <span class="sc">+</span> labs_alt <span class="sc">+</span> labs_ggt <span class="sc">+</span> labs_ldl <span class="sc">+</span> labs_triglycerides <span class="sc">+</span> labs_hdl <span class="sc">+</span> labs_quick_index, <span class="at">data =</span> pha_redcap)</span>
<span id="cb20-2"><a href="#cb20-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb20-3"><a href="#cb20-3" aria-hidden="true" tabindex="-1"></a><span class="fu">summary</span>(pha_1)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>Linear mixed model fit by REML. t-tests use Satterthwaite's method [
lmerModLmerTest]
Formula: 
phase_angle ~ (1 | record_id) + visit + allocation_group + completed_intervention +  
    duration_difference + age + sex + hypertension + hypercholesterolemia +  
    hypertrigliceridemia + drugs_w_loss + drugs_w_gain + mean_bp_mean +  
    handgrip + evs_score + alcohol_dose + kcal + labs_crp + labs_alt +  
    labs_ggt + labs_ldl + labs_triglycerides + labs_hdl + labs_quick_index
   Data: pha_redcap

REML criterion at convergence: 267.6

Scaled residuals: 
     Min       1Q   Median       3Q      Max 
-1.56730 -0.34461 -0.02411  0.32517  2.31790 

Random effects:
 Groups    Name        Variance Std.Dev.
 record_id (Intercept) 0.7246   0.8513  
 Residual              0.1072   0.3274  
Number of obs: 111, groups:  record_id, 73

Fixed effects:
                          Estimate Std. Error       df t value Pr(&gt;|t|)    
(Intercept)                6.80702    1.14545 54.72940   5.943 2.02e-07 ***
visit                      0.04367    0.05999 39.61741   0.728 0.470955    
allocation_groupGrupo B   -0.08294    0.23322 58.16739  -0.356 0.723419    
completed_interventionSim  0.13431    0.29025 62.80942   0.463 0.645154    
duration_difference        0.06796    0.07531 35.47525   0.902 0.372926    
age                       -0.15807    0.13498 60.83001  -1.171 0.246148    
sexMasculino               0.09460    0.47396 83.73697   0.200 0.842285    
hypertension1             -0.41606    0.30355 64.58323  -1.371 0.175222    
hypercholesterolemia1      0.19062    0.27727 69.89339   0.687 0.494049    
hypertrigliceridemia1     -0.34038    0.27433 76.54222  -1.241 0.218477    
drugs_w_loss1             -0.25530    0.26664 55.83333  -0.957 0.342447    
drugs_w_gain1             -0.66202    0.56551 57.08713  -1.171 0.246602    
mean_bp_mean               0.18737    0.09217 82.46459   2.033 0.045275 *  
handgrip                   0.02906    0.12915 84.31928   0.225 0.822503    
evs_score                  0.02820    0.06771 51.41295   0.417 0.678762    
alcohol_dose               0.25024    0.06910 55.31084   3.621 0.000637 ***
kcal                       0.08493    0.07469 64.71213   1.137 0.259686    
labs_crp                   0.12545    0.05608 35.78386   2.237 0.031613 *  
labs_alt                  -0.07563    0.08133 60.52700  -0.930 0.356110    
labs_ggt                   0.02983    0.10259 78.08587   0.291 0.771980    
labs_ldl                  -0.11607    0.09391 52.71421  -1.236 0.221990    
labs_triglycerides         0.04481    0.07352 35.85978   0.610 0.545998    
labs_hdl                  -0.02374    0.08890 81.65491  -0.267 0.790104    
labs_quick_index          -1.07713    3.09369 48.37580  -0.348 0.729223    
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>
Correlation matrix not shown by default, as p = 24 &gt; 12.
Use print(x, correlation=TRUE)  or
    vcov(x)        if you need it</code></pre>
</div>
</div>
<p>A modelagem estatística foi conduzida por meio de um modelo linear misto, com intercepto aleatório por participante (record_id), ajustado pelo método de máxima verossimilhança restrita (REML). O desfecho avaliado foi o ângulo de fase, e todas as variáveis contínuas foram previamente padronizadas (média = 0; desvio padrão = 1), com exceção do índice QUICKI, mantido em sua unidade original para favorecer a interpretabilidade clínica. A variável duration_difference representa o desvio absoluto (em dias) da duração planejada da intervenção (90 dias), sendo ajustada para zero na linha de base e posteriormente padronizada.</p>
<p>O modelo incluiu 24 preditores fixos e apresentou bom ajuste, com critério REML igual a 267,6 [um valor inferior ao do modelo anterior (REML = 275)], indicando melhora na parcimônia e qualidade do ajuste após a redução de variáveis.</p>
<p><strong>Efeitos fixos</strong></p>
<p>Três variáveis demonstraram associação estatisticamente significativa com o desfecho:</p>
<ul>
<li>Pressão arterial média (mean_bp_mean): apresentou associação positiva com o ângulo de fase (β = 0,19; p = 0,045), sugerindo que níveis mais elevados de pressão arterial podem estar relacionados a melhor integridade funcional da membrana celular.</li>
<li>Dose de álcool (alcohol_dose): foi o preditor com maior significância estatística (β = 0,25; p &lt; 0,001), com associação positiva ao ângulo de fase. Este achado deve ser interpretado com cautela, podendo refletir fatores comportamentais, sociais ou nutricionais não diretamente capturados no modelo.</li>
<li>Proteína C reativa (labs_crp): demonstrou associação positiva marginalmente significativa (β = 0,13; p = 0,032), o que pode sugerir uma relação complexa entre inflamação subclínica e alterações na composição ou integridade celular.</li>
</ul>
<p>As demais variáveis — incluindo visit, allocation_group, completed_intervention, duration_difference, além de variáveis laboratoriais como labs_ldl, labs_triglycerides e labs_quick_index — não apresentaram associações estatisticamente significativas com o desfecho.</p>
<p><strong>Efeitos aleatórios</strong></p>
<p>A variância associada ao intercepto aleatório por participante foi de 0,72, enquanto a variância residual foi de 0,11. Esse achado demonstra que há considerável variabilidade entre os indivíduos, reforçando a pertinência da inclusão do componente aleatório no modelo. Em outras palavras, parte substancial da variabilidade total no ângulo de fase é explicada por diferenças interindividuais, o que justifica o uso de modelos mistos para melhor estimar os efeitos fixos, controlando tais variações basais.</p>
<p><strong>Considerações metodológicas</strong></p>
<ol type="1">
<li>Multicolinearidade: Dada a inclusão de 24 preditores em um modelo com 111 observações, recomenda-se avaliar a presença de colinearidade, especialmente entre variáveis clínicas e laboratoriais potencialmente redundantes. Sugere-se a aplicação da função performance::check_collinearity() para investigar VIFs elevados, que podem inflar erros padrão e comprometer a interpretação dos coeficientes.</li>
<li>Redução de preditores: A elevada razão entre o número de preditores e o tamanho da amostra (p &gt; n/4) pode levar a sobreajuste (overfitting) e instabilidade nos coeficientes. Uma abordagem sugerida é a redução guiada por critérios de informação (AIC) ou penalização via regressão LASSO (glmnet), com o objetivo de selecionar subconjuntos mais estáveis de variáveis.</li>
<li>Avaliação da qualidade do ajuste: Recomenda-se utilizar a função performance::check_model() para verificar a normalidade dos resíduos, homocedasticidade e presença de observações influentes. Além disso, a obtenção dos valores de R² marginal e condicional pode auxiliar na compreensão da proporção da variância explicada pelo modelo fixo e pelo modelo completo (incluindo os efeitos aleatórios).</li>
<li>Tratamento da variável visit: Embora visit não tenha apresentado significância estatística no modelo atual, trata-se de uma variável importante, dado o delineamento longitudinal do estudo. Deve-se avaliar se o uso de visit como variável contínua linear é adequado ou se uma estrutura categórica com interação (e.g., visit*allocation_group) ou a inclusão de um termo de inclinação aleatória por visita ((visit | record_id)) poderia capturar melhor as trajetórias individuais ao longo do tempo.</li>
</ol>
<p><strong>Avaliação da Multicolinearidade</strong></p>
<p>Dada a inclusão de um número elevado de preditores no modelo (n = 24), avaliou-se a presença de multicolinearidade utilizando o Fator de Inflação da Variância (VIF) por meio da função performance::check_collinearity().</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb23"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb23-1"><a href="#cb23-1" aria-hidden="true" tabindex="-1"></a>performance<span class="sc">::</span><span class="fu">check_collinearity</span>(pha_1)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code># Check for Multicollinearity

Low Correlation

                   Term  VIF   VIF 95% CI Increased SE Tolerance
                  visit 2.64 [2.13, 3.38]         1.63      0.38
       allocation_group 1.24 [1.10, 1.57]         1.11      0.81
 completed_intervention 1.54 [1.31, 1.93]         1.24      0.65
    duration_difference 2.07 [1.71, 2.63]         1.44      0.48
                    age 1.67 [1.40, 2.10]         1.29      0.60
                    sex 2.41 [1.96, 3.07]         1.55      0.41
           hypertension 1.51 [1.29, 1.90]         1.23      0.66
   hypercholesterolemia 1.64 [1.39, 2.07]         1.28      0.61
   hypertrigliceridemia 1.71 [1.44, 2.15]         1.31      0.58
           drugs_w_loss 1.20 [1.08, 1.54]         1.10      0.83
           drugs_w_gain 1.13 [1.03, 1.52]         1.06      0.88
           mean_bp_mean 1.84 [1.53, 2.32]         1.36      0.54
               handgrip 1.94 [1.61, 2.46]         1.39      0.51
              evs_score 1.31 [1.15, 1.65]         1.14      0.76
           alcohol_dose 1.48 [1.27, 1.85]         1.22      0.68
                   kcal 1.67 [1.41, 2.10]         1.29      0.60
               labs_crp 1.22 [1.09, 1.56]         1.10      0.82
               labs_alt 1.48 [1.27, 1.85]         1.22      0.68
               labs_ggt 1.65 [1.39, 2.07]         1.28      0.61
               labs_ldl 1.69 [1.42, 2.12]         1.30      0.59
     labs_triglycerides 1.48 [1.27, 1.86]         1.22      0.67
               labs_hdl 1.32 [1.15, 1.66]         1.15      0.76
       labs_quick_index 1.23 [1.09, 1.56]         1.11      0.82
 Tolerance 95% CI
     [0.30, 0.47]
     [0.64, 0.91]
     [0.52, 0.76]
     [0.38, 0.59]
     [0.48, 0.71]
     [0.33, 0.51]
     [0.53, 0.77]
     [0.48, 0.72]
     [0.46, 0.70]
     [0.65, 0.93]
     [0.66, 0.97]
     [0.43, 0.65]
     [0.41, 0.62]
     [0.61, 0.87]
     [0.54, 0.79]
     [0.48, 0.71]
     [0.64, 0.92]
     [0.54, 0.79]
     [0.48, 0.72]
     [0.47, 0.70]
     [0.54, 0.79]
     [0.60, 0.87]
     [0.64, 0.92]</code></pre>
</div>
<div class="sourceCode cell-code" id="cb25"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb25-1"><a href="#cb25-1" aria-hidden="true" tabindex="-1"></a><span class="co"># r2(pha_1)  # Marginal and conditional R²</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>Resultados:</p>
<p>Todos os preditores apresentaram VIFs inferiores a 3, indicando ausência de multicolinearidade preocupante. Os valores variaram entre 1,13 e 2,64, o que representa correlações lineares fracas a moderadas entre os preditores, sem risco aparente de distorções nas estimativas dos coeficientes.</p>
<p>Abaixo, apresenta-se a interpretação convencional dos valores de VIF:</p>
<table class="caption-top table">
<colgroup>
<col style="width: 25%">
<col style="width: 74%">
</colgroup>
<thead>
<tr class="header">
<th><strong>Valor de VIF</strong></th>
<th><strong>Interpretação</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>1</td>
<td>Ausência de correlação com outras variáveis</td>
</tr>
<tr class="even">
<td>1–2</td>
<td>Correlação baixa — sem motivo de preocupação</td>
</tr>
<tr class="odd">
<td>2–5</td>
<td>Correlação moderada — acompanhar com atenção</td>
</tr>
<tr class="even">
<td>5–10</td>
<td>Correlação elevada — risco de instabilidade</td>
</tr>
<tr class="odd">
<td>&gt; 10</td>
<td>Multicolinearidade severa — intervenção necessária</td>
</tr>
</tbody>
</table>
<ol start="2" type="1">
<li>Redução do Modelo</li>
</ol>
<p>O modelo atual inclui 24 preditores fixos, o que, diante de uma amostra com 111 observações, pode comprometer o poder estatístico e a interpretabilidade dos resultados devido ao risco de sobreajuste (overfitting).</p>
<p>A escolha da melhor abordagem para simplificar o modelo depende do objetivo central da análise:</p>
<ol type="a">
<li>Modelos orientados por desempenho (predição)</li>
</ol>
<p>Caso o foco seja a performance preditiva, o uso de estratégias orientadas por dados é mais apropriado:</p>
<ul>
<li><p>Seleção stepwise (backward) com base em critérios de informação como AIC ou BIC. O AIC favorece modelos com melhor ajuste (ainda que mais complexos), enquanto o BIC impõe maior penalidade à complexidade, favorecendo modelos mais parcimoniosos.</p></li>
<li><p>Regressão penalizada via LASSO (glmnet), que tende a selecionar subconjuntos estáveis de variáveis ao impor penalizações sobre os coeficientes, sendo especialmente útil em situações com colinearidade moderada e alta dimensionalidade de preditores.</p></li>
</ul>
<ol start="2" type="a">
<li>Modelos orientados por inferência e explicação</li>
</ol>
<p>Quando o objetivo principal é a interpretação causal ou explicativa, como em estudos clínicos e intervenções, a estratégia mais robusta é a redução teórica guiada por plausibilidade clínica. Essa abordagem inclui:</p>
<ul>
<li><p>Remoção de variáveis com baixa significância estatística e sem fundamento teórico forte;</p></li>
<li><p>Exclusão de variáveis redundantes ou fortemente correlacionadas (ex.: manter labs_quick_index e excluir glicemia/insulina);</p></li>
<li><p>Atenção a comportamentos instáveis dos coeficientes — por exemplo, quando a direção (sinal) de um coeficiente muda com a inclusão de outras variáveis, indicando colinearidade ou sobreposição de efeitos.</p></li>
</ul>
<ol start="3" type="a">
<li>Estratégia híbrida recomendada</li>
</ol>
<p>Uma estratégia combinada, adequada ao contexto de ensaio clínico com múltiplas covariáveis, pode envolver: 1. Fixar um conjunto mínimo de variáveis-chave (ex.: idade, sexo, grupo de intervenção, tempo); 2. Aplicar redução stepwise ou LASSO nas demais covariáveis; 3. Comparar modelos reduzidos com o modelo completo utilizando ANOVA (anova()), gráficos de resíduos e critérios de ajuste (REML, AIC, BIC), assegurando que a simplificação não comprometa o desempenho do modelo.</p>
<ol start="3" type="1">
<li>Avaliação do Ajuste do Modelo</li>
</ol>
<p>A verificação dos pressupostos estatísticos do modelo pha_1 foi realizada com a função performance::check_model(). Os principais achados foram:</p>
<ul>
<li><p>Ajuste geral: Os gráficos de verificação preditiva (posterior predictive checks) mostraram boa sobreposição entre os valores observados e os preditos, indicando adequação geral do modelo.</p></li>
<li><p>Resíduos vs.&nbsp;valores ajustados: Foi observado leve desvio da horizontalidade nas faixas mais altas do desfecho, sugerindo possível violação da linearidade para valores elevados do ângulo de fase.</p></li>
<li><p>Homoscedasticidade: Detectou-se aumento da variância dos resíduos com o aumento dos valores preditos, caracterizando uma leve violação da homocedasticidade.</p></li>
<li><p>Normalidade dos resíduos: Os resíduos apresentaram distribuição aproximadamente normal, com pequenas assimetrias nas caudas — aceitáveis em modelos mistos com amostras de tamanho moderado.</p></li>
<li><p>Influência de observações: Algumas observações apresentaram leve influência (IDs 10, 69, 70, 75, 109), mas nenhuma ultrapassou os limiares críticos para alavancagem ou resíduos padronizados extremos.</p></li>
<li><p>Colinearidade: Todos os VIFs estiveram abaixo de 3, indicando ausência de multicolinearidade significativa.</p></li>
<li><p>Efeitos aleatórios: A distribuição dos interceptos aleatórios (por record_id) se aproximou da normalidade, reforçando a adequação da estrutura hierárquica adotada.</p></li>
</ul>
<ol start="5" type="1">
<li>Refinamento da Variável visit</li>
</ol>
<p>Dado o delineamento longitudinal do estudo, a variável visit merece atenção especial. No modelo atual, visit foi incluída como efeito fixo com estrutura linear. No entanto, isso pode ser subótimo caso o efeito do tempo varie entre participantes.</p>
<p>Tentativas de incorporar uma inclinação aleatória por visit resultaram em:</p>
<pre><code>lmer(phase_angle ~ visit + (visit | record_id), data = ...)
# Erro: número de observações (=111) ≤ número de efeitos aleatórios (=146)</code></pre>
<p>Isso indica que há insuficiência de dados para estimar uma estrutura tão complexa. A alternativa:</p>
<pre><code>lmer(phase_angle ~ visit + (1 + visit || record_id), data = ...)
# Resultado: ajuste singular (boundary (singular) fit)</code></pre>
<p>Esse ajuste singular significa que o modelo não consegue estimar a variância da inclinação aleatória para visit de forma confiável, sugerindo ausência de variação individual suficiente ou tamanho amostral limitado para suportar tal estrutura.</p>
<p>Conclusão:</p>
<p>No momento, manter visit como efeito fixo linear é estatisticamente mais robusto. Caso haja interesse em investigar mudanças específicas ao longo das visitas, uma abordagem alternativa seria tratar visit como variável categórica e incluir interações com allocation_group, por exemplo:</p>
<pre><code>phase_angle ~ allocation_group * factor(visit) + ...</code></pre>
<p>Essa estrutura permite verificar se a evolução do ângulo de fase ao longo do tempo difere entre os grupos, sem sobrecarregar o modelo com componentes aleatórios instáveis.</p>
</section>
<section id="modelo-2" class="level2">
<h2 class="anchored" data-anchor-id="modelo-2">Modelo 2</h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb29"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb29-1"><a href="#cb29-1" aria-hidden="true" tabindex="-1"></a>pha_2 <span class="ot">&lt;-</span> <span class="fu">lmer</span>(phase_angle <span class="sc">~</span> (<span class="dv">1</span> <span class="sc">|</span> record_id) <span class="sc">+</span> visit <span class="sc">+</span> allocation_group <span class="sc">+</span> age <span class="sc">+</span> sex <span class="sc">+</span> bmi <span class="sc">+</span> mean_bp_mean <span class="sc">+</span> handgrip <span class="sc">+</span> evs_score <span class="sc">+</span> kcal <span class="sc">+</span> labs_crp <span class="sc">+</span> labs_alt <span class="sc">+</span> labs_ggt <span class="sc">+</span> labs_ldl <span class="sc">+</span> labs_triglycerides <span class="sc">+</span> labs_hdl <span class="sc">+</span> labs_quick_index, <span class="at">data =</span> pha_redcap)</span>
<span id="cb29-2"><a href="#cb29-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb29-3"><a href="#cb29-3" aria-hidden="true" tabindex="-1"></a><span class="fu">summary</span>(pha_2)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>Linear mixed model fit by REML. t-tests use Satterthwaite's method [
lmerModLmerTest]
Formula: phase_angle ~ (1 | record_id) + visit + allocation_group + age +  
    sex + bmi + mean_bp_mean + handgrip + evs_score + kcal +  
    labs_crp + labs_alt + labs_ggt + labs_ldl + labs_triglycerides +  
    labs_hdl + labs_quick_index
   Data: pha_redcap

REML criterion at convergence: 275.6

Scaled residuals: 
    Min      1Q  Median      3Q     Max 
-2.1820 -0.3023 -0.0278  0.2559  3.6054 

Random effects:
 Groups    Name        Variance Std.Dev.
 record_id (Intercept) 0.5811   0.7623  
 Residual              0.1780   0.4219  
Number of obs: 111, groups:  record_id, 73

Fixed effects:
                         Estimate Std. Error        df t value Pr(&gt;|t|)    
(Intercept)              7.082490   1.176374 77.136367   6.021 5.51e-08 ***
visit                   -0.004178   0.060460 47.468510  -0.069   0.9452    
allocation_groupGrupo B -0.252382   0.214787 64.288710  -1.175   0.2443    
age                     -0.187470   0.105337 65.806508  -1.780   0.0797 .  
sexMasculino             0.335406   0.456188 84.261799   0.735   0.4642    
bmi                      0.010733   0.097621 89.559042   0.110   0.9127    
mean_bp_mean             0.174320   0.096937 93.792525   1.798   0.0753 .  
handgrip                 0.077031   0.130670 92.345158   0.590   0.5570    
evs_score                0.035996   0.073279 74.994735   0.491   0.6247    
kcal                     0.026568   0.080627 77.299814   0.330   0.7427    
labs_crp                 0.080876   0.065164 51.402275   1.241   0.2202    
labs_alt                -0.001230   0.085298 88.062468  -0.014   0.9885    
labs_ggt                 0.026733   0.096885 82.285184   0.276   0.7833    
labs_ldl                -0.064943   0.090663 92.475432  -0.716   0.4756    
labs_triglycerides       0.015440   0.079832 62.011443   0.193   0.8473    
labs_hdl                 0.002898   0.092120 93.997884   0.031   0.9750    
labs_quick_index        -1.886146   3.435802 75.554696  -0.549   0.5846    
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>
Correlation matrix not shown by default, as p = 17 &gt; 12.
Use print(x, correlation=TRUE)  or
    vcov(x)        if you need it</code></pre>
</div>
</div>
<blockquote class="blockquote">
<p>AIC(pha_1, pha_2) df AIC pha_1 26 319.5554 pha_2 19 313.6283</p>
</blockquote>
<blockquote class="blockquote">
<p>BIC(pha_1, pha_2) df BIC pha_1 26 390.0032 pha_2 19 365.1094</p>
</blockquote>
<p>A comparação entre os modelos pelo critério de informação de Akaike (AIC) e o critério bayesiano de informação (BIC) favoreceu o modelo reduzido (pha_2), que apresentou valores mais baixos de AIC (313,6 vs.&nbsp;319,6) e BIC (365,1 vs.&nbsp;390,0) em relação ao modelo completo (pha_1). Isso sugere que a simplificação do modelo resultou em melhor equilíbrio entre ajuste e complexidade, mesmo com a perda de significância estatística de algumas covariáveis previamente relevantes. No entanto, várias variáveis de relevância clínica foram eliminadas neste modelo. Por este motivo, um terceiro modelo foi criado, incluindo as variáveis de relevância clínica que não foram incluídas no modelo 2.</p>
</section>
<section id="modelo-3" class="level2">
<h2 class="anchored" data-anchor-id="modelo-3">Modelo 3</h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb32"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb32-1"><a href="#cb32-1" aria-hidden="true" tabindex="-1"></a>pha_3 <span class="ot">&lt;-</span> <span class="fu">lmer</span>(phase_angle <span class="sc">~</span> (<span class="dv">1</span> <span class="sc">|</span> record_id) <span class="sc">+</span> visit <span class="sc">+</span> allocation_group <span class="sc">+</span> completed_intervention <span class="sc">+</span> duration_difference <span class="sc">+</span> age <span class="sc">+</span> sex <span class="sc">+</span> hypertension <span class="sc">+</span> hypercholesterolemia <span class="sc">+</span> hypertrigliceridemia <span class="sc">+</span> drugs_w_loss <span class="sc">+</span> drugs_w_gain <span class="sc">+</span> mean_bp_mean <span class="sc">+</span> handgrip <span class="sc">+</span> evs_score <span class="sc">+</span> alcohol_dose <span class="sc">+</span> kcal <span class="sc">+</span> labs_crp <span class="sc">+</span> labs_ldl <span class="sc">+</span> labs_quick_index, <span class="at">data =</span> pha_redcap)</span>
<span id="cb32-2"><a href="#cb32-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb32-3"><a href="#cb32-3" aria-hidden="true" tabindex="-1"></a><span class="fu">summary</span>(pha_3)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>Linear mixed model fit by REML. t-tests use Satterthwaite's method [
lmerModLmerTest]
Formula: 
phase_angle ~ (1 | record_id) + visit + allocation_group + completed_intervention +  
    duration_difference + age + sex + hypertension + hypercholesterolemia +  
    hypertrigliceridemia + drugs_w_loss + drugs_w_gain + mean_bp_mean +  
    handgrip + evs_score + alcohol_dose + kcal + labs_crp + labs_ldl +  
    labs_quick_index
   Data: pha_redcap

REML criterion at convergence: 256.2

Scaled residuals: 
     Min       1Q   Median       3Q      Max 
-1.67542 -0.37054 -0.01038  0.28131  2.47511 

Random effects:
 Groups    Name        Variance Std.Dev.
 record_id (Intercept) 0.6800   0.8246  
 Residual              0.1089   0.3300  
Number of obs: 111, groups:  record_id, 73

Fixed effects:
                          Estimate Std. Error       df t value Pr(&gt;|t|)    
(Intercept)                6.73199    1.12618 62.54202   5.978 1.18e-07 ***
visit                      0.04629    0.05831 39.70014   0.794  0.43198    
allocation_groupGrupo B   -0.08824    0.22558 60.66392  -0.391  0.69704    
completed_interventionSim  0.09468    0.27813 63.80928   0.340  0.73466    
duration_difference        0.06221    0.07418 37.43891   0.839  0.40701    
age                       -0.14627    0.13082 63.16447  -1.118  0.26776    
sexMasculino               0.05360    0.44022 84.14001   0.122  0.90339    
hypertension1             -0.37768    0.29365 65.31066  -1.286  0.20294    
hypercholesterolemia1      0.13976    0.26480 72.33121   0.528  0.59926    
hypertrigliceridemia1     -0.21111    0.23292 62.70194  -0.906  0.36822    
drugs_w_loss1             -0.25216    0.25896 58.72682  -0.974  0.33419    
drugs_w_gain1             -0.64500    0.54712 60.02599  -1.179  0.24308    
mean_bp_mean               0.18141    0.08674 82.54798   2.092  0.03955 *  
handgrip                   0.04801    0.12342 88.77623   0.389  0.69823    
evs_score                  0.03655    0.06602 54.80914   0.554  0.58207    
alcohol_dose               0.24846    0.06783 59.87919   3.663  0.00053 ***
kcal                       0.09010    0.07346 72.69290   1.227  0.22393    
labs_crp                   0.11992    0.05502 40.88103   2.180  0.03509 *  
labs_ldl                  -0.09732    0.08787 60.12632  -1.108  0.27247    
labs_quick_index          -0.90144    3.03633 57.49316  -0.297  0.76762    
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>
Correlation matrix not shown by default, as p = 20 &gt; 12.
Use print(x, correlation=TRUE)  or
    vcov(x)        if you need it</code></pre>
</div>
<div class="sourceCode cell-code" id="cb35"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb35-1"><a href="#cb35-1" aria-hidden="true" tabindex="-1"></a><span class="fu">check_collinearity</span>(pha_3)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code># Check for Multicollinearity

Low Correlation

                   Term  VIF   VIF 95% CI Increased SE Tolerance
                  visit 2.46 [1.98, 3.18]         1.57      0.41
       allocation_group 1.22 [1.08, 1.58]         1.11      0.82
 completed_intervention 1.49 [1.27, 1.89]         1.22      0.67
    duration_difference 1.99 [1.63, 2.54]         1.41      0.50
                    age 1.65 [1.39, 2.11]         1.29      0.60
                    sex 2.20 [1.79, 2.83]         1.48      0.45
           hypertension 1.50 [1.27, 1.90]         1.22      0.67
   hypercholesterolemia 1.58 [1.34, 2.02]         1.26      0.63
   hypertrigliceridemia 1.30 [1.14, 1.67]         1.14      0.77
           drugs_w_loss 1.20 [1.07, 1.57]         1.10      0.83
           drugs_w_gain 1.12 [1.03, 1.56]         1.06      0.89
           mean_bp_mean 1.65 [1.39, 2.10]         1.29      0.61
               handgrip 1.84 [1.52, 2.35]         1.36      0.54
              evs_score 1.25 [1.11, 1.62]         1.12      0.80
           alcohol_dose 1.44 [1.23, 1.83]         1.20      0.69
                   kcal 1.62 [1.36, 2.06]         1.27      0.62
               labs_crp 1.17 [1.05, 1.55]         1.08      0.86
               labs_ldl 1.50 [1.28, 1.91]         1.23      0.67
       labs_quick_index 1.19 [1.07, 1.56]         1.09      0.84
 Tolerance 95% CI
     [0.31, 0.51]
     [0.63, 0.92]
     [0.53, 0.79]
     [0.39, 0.61]
     [0.48, 0.72]
     [0.35, 0.56]
     [0.53, 0.78]
     [0.50, 0.75]
     [0.60, 0.88]
     [0.64, 0.93]
     [0.64, 0.98]
     [0.48, 0.72]
     [0.42, 0.66]
     [0.62, 0.90]
     [0.55, 0.81]
     [0.48, 0.73]
     [0.65, 0.95]
     [0.52, 0.78]
     [0.64, 0.94]</code></pre>
</div>
<div class="sourceCode cell-code" id="cb37"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb37-1"><a href="#cb37-1" aria-hidden="true" tabindex="-1"></a><span class="fu">AIC</span>(pha_1, pha_2, pha_3)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>      df      AIC
pha_1 26 319.5554
pha_2 19 313.6283
pha_3 22 300.2185</code></pre>
</div>
<div class="sourceCode cell-code" id="cb39"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb39-1"><a href="#cb39-1" aria-hidden="true" tabindex="-1"></a><span class="fu">BIC</span>(pha_1, pha_2, pha_3)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>      df      BIC
pha_1 26 390.0032
pha_2 19 365.1094
pha_3 22 359.8281</code></pre>
</div>
<div class="sourceCode cell-code" id="cb41"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb41-1"><a href="#cb41-1" aria-hidden="true" tabindex="-1"></a><span class="fu">r2</span>(pha_3)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code># R2 for Mixed Models

  Conditional R2: 0.896
     Marginal R2: 0.250</code></pre>
</div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb43"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb43-1"><a href="#cb43-1" aria-hidden="true" tabindex="-1"></a>pha_3_plots <span class="ot">&lt;-</span> performance<span class="sc">::</span><span class="fu">check_model</span>(pha_3)</span>
<span id="cb43-2"><a href="#cb43-2" aria-hidden="true" tabindex="-1"></a><span class="fu">print</span>(pha_3_plots)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="Results_main_outcomes_files/figure-html/pha_3_check-model-1.png" class="img-fluid figure-img" width="960"></p>
</figure>
</div>
</div>
</div>
<p><strong>Comparação entre Modelos e Avaliação do Ajuste Final</strong></p>
<p>Três modelos hierárquicos foram comparados utilizando os critérios de informação de Akaike (AIC) e Bayesiano (BIC). O modelo pha_3, contendo 22 preditores fixos e intercepto aleatório por participante (record_id), apresentou os menores valores de AIC (300,2) e BIC (359,8), superando o modelo completo pha_1 (AIC = 319,6; BIC = 390,0) e o modelo intermediário pha_2 (AIC = 313,6; BIC = 365,1). Esses resultados indicam que pha_3 representa o melhor equilíbrio entre qualidade de ajuste e parcimônia, sendo selecionado como modelo final para interpretação dos achados.</p>
<p>A construção do modelo pha_3 foi guiada por critérios estatísticos e clínicos, priorizando variáveis com significância nas versões anteriores, plausibilidade teórica e estabilidade das estimativas. O modelo manteve 22 preditores e apresentou um critério de máxima verossimilhança restrita (REML) igual a 256,2, inferior ao dos modelos anteriores, reforçando a melhoria no ajuste.</p>
<p>Entre os preditores avaliados, três variáveis mantiveram associação estatisticamente significativa com o desfecho:</p>
<ul>
<li><p>Pressão arterial média (mean_bp_mean): β = 0,18; p = 0,040</p></li>
<li><p>Dose de álcool (alcohol_dose): β = 0,25; p &lt; 0,001</p></li>
<li><p>Proteína C reativa (labs_crp): β = 0,12; p = 0,035</p></li>
</ul>
<p>Essas associações permaneceram robustas mesmo após ajuste multivariado, sugerindo que níveis mais elevados de pressão arterial, maior consumo de álcool e níveis mais altos de inflamação subclínica estão positivamente relacionados ao ângulo de fase. A variável visit foi mantida no modelo como controle temporal, apesar de não apresentar significância estatística (p = 0,432).</p>
<p>A variância do intercepto aleatório por participante permaneceu elevada (σ² = 0,68), corroborando a presença de heterogeneidade individual e justificando a estrutura mista do modelo. O R² marginal foi de 0,250, indicando que 25% da variância total do desfecho é explicada pelas variáveis fixas. Já o R² condicional foi de 0,896, refletindo a alta proporção da variância explicada ao considerar a estrutura de efeitos aleatórios, em consonância com o delineamento longitudinal do estudo.</p>
<p>A análise de multicolinearidade mostrou valores de VIF menor que três para todos os preditores, descartando problemas relevantes de colinearidade. Os diagnósticos do modelo indicaram distribuição adequada dos resíduos, ausência de observações altamente influentes e normalidade satisfatória dos efeitos aleatórios, corroborando a adequação do modelo ajustado.</p>
<p><strong>Verificação dos Pressupostos do Modelo Final</strong></p>
<p>A avaliação dos pressupostos do modelo pha_3 foi realizada com auxílio da função performance::check_model(), cujos gráficos diagnósticos estão apresentados na Figura X.</p>
<ul>
<li><p>Posterior Predictive Check: Houve boa sobreposição entre os dados observados e os valores preditos pelo modelo, sugerindo adequada capacidade preditiva.</p></li>
<li><p>Linearidade: Os resíduos apresentaram tendência de curvatura em valores mais altos do desfecho, o que pode indicar leve violação do pressuposto de linearidade, mas sem evidências de distorção grave.</p></li>
<li><p>Homoscedasticidade: Detectou-se discreto aumento da variabilidade dos resíduos em faixas mais altas dos valores ajustados, sugerindo leve heterocedasticidade.</p></li>
<li><p>Normalidade dos resíduos: A distribuição dos resíduos foi aproximadamente normal, com desvios leves nas caudas, aceitáveis para modelos mistos em amostras moderadas.</p></li>
<li><p>Observações influentes: Algumas observações foram marcadas com leve alavancagem (IDs 10, 69, 70, 75, 109), mas todas permaneceram dentro dos limites aceitáveis de influência.</p></li>
<li><p>Multicolinearidade: Todos os preditores apresentaram valores de VIF inferiores a 3, afastando preocupações com colinearidade.</p></li>
<li><p>Normalidade dos efeitos aleatórios (record_id): A distribuição dos interceptos aleatórios foi próxima da normal, com leve assimetria nas extremidades, validando a escolha do modelo com intercepto aleatório.</p></li>
</ul>
</section>
</section>
<section id="pcr" class="level1">
<h1>PCR</h1>
<section id="scaling-1" class="level2">
<h2 class="anchored" data-anchor-id="scaling-1">Scaling</h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb44"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb44-1"><a href="#cb44-1" aria-hidden="true" tabindex="-1"></a>data_crp <span class="ot">&lt;-</span> data_model <span class="sc">%&gt;%</span></span>
<span id="cb44-2"><a href="#cb44-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="fu">across</span>(</span>
<span id="cb44-3"><a href="#cb44-3" aria-hidden="true" tabindex="-1"></a>    <span class="at">.cols =</span> <span class="fu">c</span>(</span>
<span id="cb44-4"><a href="#cb44-4" aria-hidden="true" tabindex="-1"></a>        duration_difference, age,</span>
<span id="cb44-5"><a href="#cb44-5" aria-hidden="true" tabindex="-1"></a>        whoqol_score_overall, dass_score_depression, dass_score_anxiety, dass_score_stress, ecap_score, </span>
<span id="cb44-6"><a href="#cb44-6" aria-hidden="true" tabindex="-1"></a>        abdomen, bmi, mean_bp_mean,</span>
<span id="cb44-7"><a href="#cb44-7" aria-hidden="true" tabindex="-1"></a>        phase_angle, resistance, reactance, </span>
<span id="cb44-8"><a href="#cb44-8" aria-hidden="true" tabindex="-1"></a>        handgrip, evs_score, alcohol_dose,</span>
<span id="cb44-9"><a href="#cb44-9" aria-hidden="true" tabindex="-1"></a>        carbs_kcal, protein_kcal, fat_kcal, kcal,</span>
<span id="cb44-10"><a href="#cb44-10" aria-hidden="true" tabindex="-1"></a>        labs_ast, labs_alt, labs_ggt, labs_alkp,</span>
<span id="cb44-11"><a href="#cb44-11" aria-hidden="true" tabindex="-1"></a>        labs_cholesterol, labs_ldl, labs_hba1c, labs_triglycerides,</span>
<span id="cb44-12"><a href="#cb44-12" aria-hidden="true" tabindex="-1"></a>        labs_hdl, labs_glucose, labs_insulin, labs_homa_ir</span>
<span id="cb44-13"><a href="#cb44-13" aria-hidden="true" tabindex="-1"></a>    ),</span>
<span id="cb44-14"><a href="#cb44-14" aria-hidden="true" tabindex="-1"></a>    <span class="at">.fns =</span> <span class="sc">~</span> <span class="fu">as.numeric</span>(<span class="fu">scale</span>(.))</span>
<span id="cb44-15"><a href="#cb44-15" aria-hidden="true" tabindex="-1"></a>  )) <span class="sc">%&gt;%</span> </span>
<span id="cb44-16"><a href="#cb44-16" aria-hidden="true" tabindex="-1"></a>    <span class="fu">rename</span>(</span>
<span id="cb44-17"><a href="#cb44-17" aria-hidden="true" tabindex="-1"></a>        <span class="at">crp_raw =</span> labs_crp</span>
<span id="cb44-18"><a href="#cb44-18" aria-hidden="true" tabindex="-1"></a>    ) <span class="sc">%&gt;%</span> </span>
<span id="cb44-19"><a href="#cb44-19" aria-hidden="true" tabindex="-1"></a>    <span class="fu">mutate</span>(</span>
<span id="cb44-20"><a href="#cb44-20" aria-hidden="true" tabindex="-1"></a>        <span class="at">crp_log =</span> <span class="fu">log1p</span>(crp_raw),</span>
<span id="cb44-21"><a href="#cb44-21" aria-hidden="true" tabindex="-1"></a>        <span class="at">crp_log_scaled =</span> <span class="fu">as.numeric</span>(<span class="fu">scale</span>(crp_log))</span>
<span id="cb44-22"><a href="#cb44-22" aria-hidden="true" tabindex="-1"></a>    ) <span class="sc">%&gt;%</span> </span>
<span id="cb44-23"><a href="#cb44-23" aria-hidden="true" tabindex="-1"></a>    <span class="fu">relocate</span>(</span>
<span id="cb44-24"><a href="#cb44-24" aria-hidden="true" tabindex="-1"></a>        crp_log, crp_log_scaled, <span class="at">.after =</span> crp_raw</span>
<span id="cb44-25"><a href="#cb44-25" aria-hidden="true" tabindex="-1"></a>    )</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb45"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb45-1"><a href="#cb45-1" aria-hidden="true" tabindex="-1"></a>data_crp <span class="ot">&lt;-</span> data_crp <span class="sc">%&gt;%</span></span>
<span id="cb45-2"><a href="#cb45-2" aria-hidden="true" tabindex="-1"></a>  dplyr<span class="sc">::</span><span class="fu">mutate</span>(</span>
<span id="cb45-3"><a href="#cb45-3" aria-hidden="true" tabindex="-1"></a>    <span class="at">compliance_score_adjusted =</span> dplyr<span class="sc">::</span><span class="fu">case_when</span>(</span>
<span id="cb45-4"><a href="#cb45-4" aria-hidden="true" tabindex="-1"></a>      visit <span class="sc">==</span> <span class="dv">1</span> <span class="sc">~</span> <span class="dv">1</span>,</span>
<span id="cb45-5"><a href="#cb45-5" aria-hidden="true" tabindex="-1"></a>      <span class="cn">TRUE</span> <span class="sc">~</span> compliance_score_visit</span>
<span id="cb45-6"><a href="#cb45-6" aria-hidden="true" tabindex="-1"></a>    )</span>
<span id="cb45-7"><a href="#cb45-7" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb45-8"><a href="#cb45-8" aria-hidden="true" tabindex="-1"></a>  dplyr<span class="sc">::</span><span class="fu">relocate</span>(compliance_score_adjusted, <span class="at">.after =</span> compliance_score_visit)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="per-protocol" class="level2">
<h2 class="anchored" data-anchor-id="per-protocol">Per protocol</h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb46"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb46-1"><a href="#cb46-1" aria-hidden="true" tabindex="-1"></a>pcr_1 <span class="ot">&lt;-</span> <span class="fu">lmer</span>(</span>
<span id="cb46-2"><a href="#cb46-2" aria-hidden="true" tabindex="-1"></a>  crp_log <span class="sc">~</span> </span>
<span id="cb46-3"><a href="#cb46-3" aria-hidden="true" tabindex="-1"></a>      allocation_group <span class="sc">+</span> visit <span class="sc">+</span> </span>
<span id="cb46-4"><a href="#cb46-4" aria-hidden="true" tabindex="-1"></a>      duration_difference <span class="sc">+</span> <span class="co"># compliance_score_adjusted + completed_intervention </span></span>
<span id="cb46-5"><a href="#cb46-5" aria-hidden="true" tabindex="-1"></a>      (<span class="dv">1</span> <span class="sc">|</span> record_id) <span class="sc">+</span></span>
<span id="cb46-6"><a href="#cb46-6" aria-hidden="true" tabindex="-1"></a>      age <span class="sc">+</span> sex <span class="sc">+</span> hypertension <span class="sc">+</span> hypercholesterolemia <span class="sc">+</span> hypertrigliceridemia <span class="sc">+</span> drugs_w_loss <span class="sc">+</span> drugs_w_gain <span class="sc">+</span> </span>
<span id="cb46-7"><a href="#cb46-7" aria-hidden="true" tabindex="-1"></a>      bmi <span class="sc">+</span> mean_bp_mean <span class="sc">+</span> handgrip <span class="sc">+</span> evs_score <span class="sc">+</span> <span class="co"># phase angle +</span></span>
<span id="cb46-8"><a href="#cb46-8" aria-hidden="true" tabindex="-1"></a>      <span class="co"># whoqol_score_overall + dass_score_depression + dass_score_anxiety + dass_score_stress + ecap_score</span></span>
<span id="cb46-9"><a href="#cb46-9" aria-hidden="true" tabindex="-1"></a>      alcohol_dose <span class="sc">+</span> kcal <span class="sc">+</span> </span>
<span id="cb46-10"><a href="#cb46-10" aria-hidden="true" tabindex="-1"></a>      labs_alt <span class="sc">+</span> labs_ggt <span class="sc">+</span> labs_ldl <span class="sc">+</span> labs_triglycerides <span class="sc">+</span> labs_hdl <span class="sc">+</span> labs_quick_index,</span>
<span id="cb46-11"><a href="#cb46-11" aria-hidden="true" tabindex="-1"></a>  <span class="at">data =</span> data_crp</span>
<span id="cb46-12"><a href="#cb46-12" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb46-13"><a href="#cb46-13" aria-hidden="true" tabindex="-1"></a><span class="fu">summary</span>(pcr_1)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>Linear mixed model fit by REML. t-tests use Satterthwaite's method [
lmerModLmerTest]
Formula: crp_log ~ allocation_group + visit + duration_difference + (1 |  
    record_id) + age + sex + hypertension + hypercholesterolemia +  
    hypertrigliceridemia + drugs_w_loss + drugs_w_gain + bmi +  
    mean_bp_mean + handgrip + evs_score + alcohol_dose + kcal +  
    labs_alt + labs_ggt + labs_ldl + labs_triglycerides + labs_hdl +  
    labs_quick_index
   Data: data_crp

REML criterion at convergence: 263.9

Scaled residuals: 
    Min      1Q  Median      3Q     Max 
-1.9634 -0.4535  0.0329  0.4070  3.9284 

Random effects:
 Groups    Name        Variance Std.Dev.
 record_id (Intercept) 0.1954   0.4421  
 Residual              0.2934   0.5417  
Number of obs: 113, groups:  record_id, 73

Fixed effects:
                        Estimate Std. Error       df t value Pr(&gt;|t|)   
(Intercept)              3.98480    1.18979 89.14517   3.349  0.00119 **
allocation_groupGrupo B  0.04501    0.17142 55.87514   0.263  0.79386   
visit                    0.02521    0.07786 59.11390   0.324  0.74722   
duration_difference     -0.27519    0.09993 67.06910  -2.754  0.00757 **
age                     -0.05874    0.09683 61.98324  -0.607  0.54631   
sexMasculino            -0.24206    0.38972 69.66130  -0.621  0.53656   
hypertension1            0.00670    0.23211 59.28317   0.029  0.97707   
hypercholesterolemia1    0.10855    0.21615 61.23416   0.502  0.61733   
hypertrigliceridemia1    0.16259    0.22052 72.86998   0.737  0.46330   
drugs_w_loss1            0.23680    0.19022 54.02795   1.245  0.21856   
drugs_w_gain1            0.22627    0.41896 58.62749   0.540  0.59119   
bmi                      0.15183    0.07970 65.50959   1.905  0.06116 . 
mean_bp_mean            -0.05400    0.08511 76.03817  -0.634  0.52766   
handgrip                -0.06965    0.11295 70.68754  -0.617  0.53942   
evs_score                0.16757    0.07449 89.59793   2.250  0.02692 * 
alcohol_dose            -0.03274    0.07223 83.58122  -0.453  0.65156   
kcal                     0.06695    0.07874 89.36047   0.850  0.39746   
labs_alt                -0.07061    0.07966 80.04805  -0.886  0.37810   
labs_ggt                 0.06535    0.08398 75.88741   0.778  0.43886   
labs_ldl                 0.14553    0.10009 86.51472   1.454  0.14957   
labs_triglycerides      -0.09479    0.09315 80.14290  -1.018  0.31193   
labs_hdl                 0.01256    0.08298 79.71383   0.151  0.88003   
labs_quick_index        -7.89729    3.34852 88.95312  -2.358  0.02054 * 
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>
Correlation matrix not shown by default, as p = 23 &gt; 12.
Use print(x, correlation=TRUE)  or
    vcov(x)        if you need it</code></pre>
</div>
<div class="sourceCode cell-code" id="cb49"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb49-1"><a href="#cb49-1" aria-hidden="true" tabindex="-1"></a><span class="fu">check_collinearity</span>(pcr_1)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code># Check for Multicollinearity

Low Correlation

                 Term  VIF   VIF 95% CI Increased SE Tolerance Tolerance 95% CI
     allocation_group 1.35 [1.18, 1.71]         1.16      0.74     [0.59, 0.85]
                visit 1.93 [1.60, 2.45]         1.39      0.52     [0.41, 0.63]
  duration_difference 1.67 [1.41, 2.11]         1.29      0.60     [0.47, 0.71]
                  age 1.70 [1.43, 2.14]         1.30      0.59     [0.47, 0.70]
                  sex 3.26 [2.60, 4.21]         1.81      0.31     [0.24, 0.39]
         hypertension 1.87 [1.55, 2.36]         1.37      0.54     [0.42, 0.65]
 hypercholesterolemia 1.97 [1.63, 2.50]         1.40      0.51     [0.40, 0.61]
 hypertrigliceridemia 2.24 [1.83, 2.85]         1.50      0.45     [0.35, 0.55]
         drugs_w_loss 1.24 [1.10, 1.58]         1.11      0.81     [0.63, 0.91]
         drugs_w_gain 1.18 [1.06, 1.53]         1.09      0.84     [0.65, 0.94]
                  bmi 1.23 [1.09, 1.57]         1.11      0.81     [0.64, 0.91]
         mean_bp_mean 1.55 [1.32, 1.95]         1.25      0.64     [0.51, 0.76]
             handgrip 2.46 [1.99, 3.14]         1.57      0.41     [0.32, 0.50]
            evs_score 1.38 [1.19, 1.73]         1.17      0.73     [0.58, 0.84]
         alcohol_dose 1.61 [1.36, 2.03]         1.27      0.62     [0.49, 0.73]
                 kcal 1.50 [1.28, 1.89]         1.23      0.67     [0.53, 0.78]
             labs_alt 1.52 [1.30, 1.91]         1.23      0.66     [0.52, 0.77]
             labs_ggt 1.66 [1.40, 2.09]         1.29      0.60     [0.48, 0.72]
             labs_ldl 2.09 [1.72, 2.66]         1.45      0.48     [0.38, 0.58]
   labs_triglycerides 2.05 [1.69, 2.60]         1.43      0.49     [0.38, 0.59]
             labs_hdl 1.34 [1.17, 1.69]         1.16      0.75     [0.59, 0.86]
     labs_quick_index 1.38 [1.20, 1.74]         1.18      0.72     [0.57, 0.83]</code></pre>
</div>
<div class="sourceCode cell-code" id="cb51"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb51-1"><a href="#cb51-1" aria-hidden="true" tabindex="-1"></a><span class="fu">AIC</span>(pcr_1)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>[1] 313.9451</code></pre>
</div>
<div class="sourceCode cell-code" id="cb53"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb53-1"><a href="#cb53-1" aria-hidden="true" tabindex="-1"></a><span class="fu">BIC</span>(pcr_1)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>[1] 382.1297</code></pre>
</div>
<div class="sourceCode cell-code" id="cb55"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb55-1"><a href="#cb55-1" aria-hidden="true" tabindex="-1"></a><span class="fu">r2</span>(pcr_1)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code># R2 for Mixed Models

  Conditional R2: 0.576
     Marginal R2: 0.293</code></pre>
</div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb57"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb57-1"><a href="#cb57-1" aria-hidden="true" tabindex="-1"></a>pcr_1_plots <span class="ot">&lt;-</span> performance<span class="sc">::</span><span class="fu">check_model</span>(pcr_1)</span>
<span id="cb57-2"><a href="#cb57-2" aria-hidden="true" tabindex="-1"></a><span class="fu">print</span>(pcr_1_plots)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="Results_main_outcomes_files/figure-html/unnamed-chunk-13-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
<p><strong>Full Analysis of the Final Parsimonious Model</strong></p>
<ol type="1">
<li>Model Fit</li>
</ol>
<ul>
<li><strong>AIC:</strong> 313.95 (lowest among previous tested models)</li>
<li><strong>BIC:</strong> 382.13</li>
<li><strong>Conditional R²:</strong> 0.576 (variance explained by fixed + random effects)</li>
<li><strong>Marginal R²:</strong> 0.293 (variance explained by fixed effects)</li>
</ul>
<p>This is a well-balanced model with the best AIC/BIC trade-off seen so far, offering high parsimony without sacrificing explanatory power.</p>
<ol start="2" type="1">
<li>Significant Predictors</li>
</ol>
<table class="caption-top table">
<colgroup>
<col style="width: 17%">
<col style="width: 17%">
<col style="width: 17%">
<col style="width: 47%">
</colgroup>
<thead>
<tr class="header">
<th>Predictor</th>
<th>Estimate</th>
<th>p-value</th>
<th>Interpretation</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><strong>Duration difference</strong></td>
<td>-0.275</td>
<td>0.0076</td>
<td>Each SD increase in deviation from planned duration predicts a ~24% reduction in CRP (exp(-0.275) ≈ 0.76). This suggests that <strong>poor adherence is strongly associated with lower inflammation reduction</strong>.</td>
</tr>
<tr class="even">
<td><strong>EVS score</strong></td>
<td>0.168</td>
<td>0.0269</td>
<td>Surprisingly, higher physical activity correlates with <strong>higher CRP</strong>, which may warrant further investigation (possible reverse causality, measurement error, or confounding).</td>
</tr>
<tr class="odd">
<td><strong>Labs_quick_index</strong></td>
<td>-7.897</td>
<td>0.0205</td>
<td>A strong inverse association with CRP, reflecting the metabolic-inflammation relationship where better insulin sensitivity predicts lower CRP.</td>
</tr>
</tbody>
</table>
<p>Borderline - <strong>BMI:</strong> p = 0.061 — higher BMI tends to associate with higher CRP, close to significance.</p>
<ol start="3" type="1">
<li>Non-Significant Predictors</li>
</ol>
<ul>
<li>Allocation group, visit number, age, sex, hypertension, cholesterol, triglycerides, blood pressure, handgrip strength, alcohol, kcal intake, and most lab biomarkers (ALT, GGT, LDL, HDL, triglycerides) are <strong>not significant</strong>.</li>
</ul>
<p>This reinforces that <strong>adherence (duration difference)</strong> and <strong>metabolic status (quick index)</strong> are primary drivers of CRP changes, while the intervention itself (allocation_group) does not show an effect after accounting for adherence.</p>
<ol start="4" type="1">
<li>Diagnostics</li>
</ol>
<ul>
<li>Posterior Predictive Check: The model-predicted density aligns well with observed CRP log values, suggesting good distributional fit.</li>
<li>Linearity: Residuals are roughly centered but some curvature appears, indicating minor deviations from perfect linearity.</li>
<li>Homogeneity of Variance: Slight funneling with higher fitted values showing slightly more variance, but overall acceptable.</li>
<li>Influential Observations: A few leverage points (IDs 106, 59, 75, 107) exist but are within acceptable Cook’s distance boundaries.</li>
<li>Collinearity: All VIFs are low to moderate (max ~3.26 for sex), indicating no problematic collinearity.</li>
<li>Normality: Residuals are reasonably normal with some right-side tails, typical for CRP-type biomarkers.</li>
<li>Random Effects:The QQ plot of random effects shows a reasonable alignment with normality assumptions.</li>
</ul>
<ol start="5" type="1">
<li>Interpretation</li>
</ol>
<ul>
<li><p>The model confirms that <strong>adherence to the study duration (duration_difference)</strong> is a major determinant of CRP reduction, suggesting that participants who better adhered to the study protocol (in terms of time on the intervention) achieved better inflammatory outcomes.</p></li>
<li><p><strong>Insulin sensitivity (labs_quick_index)</strong> remains a key independent predictor of CRP, aligning with well-established metabolic-inflammation links.</p></li>
</ul>
<p>The unexpected positive association between <strong>EVS (physical activity)</strong> and CRP could be due to:</p>
<ul>
<li>Reverse causality (participants with higher CRP are advised to exercise more)</li>
<li>Measurement limitations of EVS</li>
<li>Confounding from unmeasured variables (e.g., acute inflammatory events or infections)</li>
</ul>
<p>The lack of direct intervention effect (<strong>allocation_group</strong>) suggests the effect operates through <strong>adherence</strong> rather than treatment allocation per se, fitting a <strong>per-protocol-like</strong> model interpretation.</p>
<ol start="6" type="1">
<li>Conclusion</li>
</ol>
<ul>
<li>The model is statistically solid, with good fit and no major assumption violations.</li>
<li>The final model captures the essence of this study: <strong>inflammatory outcomes are driven by adherence and metabolic status rather than random allocation alone</strong>.</li>
<li>This model is robust, parsimonious, and clinically interpretable.</li>
</ul>
<ol start="7" type="1">
<li>Recommendation</li>
</ol>
<ul>
<li>Report this model as the main per-protocol analysis.</li>
<li>Discuss the adherence effect in the context of real-world clinical applicability.</li>
<li>Consider a sensitivity analysis exploring the unexpected positive EVS-CRP relationship.</li>
</ul>
<section id="heteroscedasticity" class="level3">
<h3 class="anchored" data-anchor-id="heteroscedasticity">Heteroscedasticity</h3>
<p>Heteroscedasticity is a violation of linear mixed model assumptions, and we’re already applying a log transformation on the outcome (log1p(crp)) plus standardizing all numeric predictors. When log-transforming the outcome is not fully sufficient, it indicates either residual skewness, heavy tails, or that the Gaussian error assumption doesn’t capture the data-generating process well.</p>
<p><em>Modeling strategies to address this issue:</em></p>
<p><strong>Generalized Linear Mixed Model (GLMM) with Non-Gaussian Distribution</strong></p>
<p>Given the characteristics of your outcome variable (<strong>CRP</strong>, which is strictly positive, right-skewed, and heteroscedastic), moving from a Gaussian-based LMM to a <strong>Generalized Linear Mixed Model (GLMM)</strong> with an appropriate distribution is both statistically justified and likely to yield better model fit.</p>
<ol type="1">
<li>Theoretical Justification</li>
</ol>
<ul>
<li><strong>CRP is a continuous, positive, right-skewed variable.</strong></li>
<li>Log-transforming CRP partially addresses skewness but doesn’t fully solve: Heteroscedasticity (as your diagnostic plots show); The misspecification from assuming normal residuals.</li>
<li>GLMM allows you to model CRP with a distribution that reflects its natural properties rather than forcing it into Gaussian assumptions.</li>
</ul>
<ol start="2" type="1">
<li>Addressing Heteroscedasticity</li>
</ol>
<ul>
<li>Your residual plots still show <strong>variance increasing with fitted values</strong> — a classic signal that a Gaussian model is a poor fit for the scale of your data.</li>
<li>GLMMs inherently model mean-variance relationships correctly for non-normal distributions.</li>
</ul>
<ol start="3" type="1">
<li>Precedent in Biomedical Literature. Biomarkers like CRP are frequently modeled with:</li>
</ol>
<ul>
<li><strong>Gamma distribution with log link</strong> (most common for positive, skewed continuous outcomes).</li>
<li><strong>Inverse Gaussian</strong> (alternative when the data have very heavy tails).</li>
<li>This approach is standard in fields like clinical epidemiology, pharmacometrics, and biomarker research.</li>
</ul>
<p>Practical Advantages. Coefficients remain on the <strong>log scale</strong>, similar to your LMM with log(CRP), but:</p>
<ul>
<li>You model CRP in its native distribution rather than relying on transformation.</li>
<li><strong>Heteroscedasticity becomes irrelevant</strong>, because Gamma models naturally assume variance increases with the mean.</li>
<li>Interpretation becomes <strong>multiplicative</strong>, which is often more intuitive for clinicians and health scientists (e.g., “a 20% increase in CRP per unit change in X”).</li>
</ul>
<p>Potential Models to Fit - <code>glmer()</code> from <strong>lme4</strong> - <code>glmmTMB()</code> from <strong>glmmTMB</strong> — more flexible and often preferred for Gamma models.</p>
<p>If the log-transformed CRP still shows heteroscedasticity, the Gaussian model may be inherently misspecified. A GLMM can explicitly model the distribution of the outcome with an appropriate family. Candidates:</p>
<ul>
<li>Gamma GLMM: For strictly positive, right-skewed outcomes like CRP.</li>
<li>Inverse Gaussian GLMM: Also for skewed continuous data, with heavier tails.</li>
<li>Log-normal GLMM: Another valid approach when a log transformation stabilizes variance but doesn’t fully normalize residuals.</li>
</ul>
<section id="glmm" class="level4">
<h4 class="anchored" data-anchor-id="glmm">GLMM</h4>
<div class="cell">
<div class="sourceCode cell-code" id="cb58"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb58-1"><a href="#cb58-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(glmmTMB)</span>
<span id="cb58-2"><a href="#cb58-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb58-3"><a href="#cb58-3" aria-hidden="true" tabindex="-1"></a>pcr_glmm <span class="ot">&lt;-</span> <span class="fu">glmmTMB</span>(</span>
<span id="cb58-4"><a href="#cb58-4" aria-hidden="true" tabindex="-1"></a>  crp_raw <span class="sc">~</span> </span>
<span id="cb58-5"><a href="#cb58-5" aria-hidden="true" tabindex="-1"></a>      allocation_group <span class="sc">+</span> visit <span class="sc">+</span> duration_difference <span class="sc">+</span> </span>
<span id="cb58-6"><a href="#cb58-6" aria-hidden="true" tabindex="-1"></a>      (<span class="dv">1</span> <span class="sc">|</span> record_id) <span class="sc">+</span> </span>
<span id="cb58-7"><a href="#cb58-7" aria-hidden="true" tabindex="-1"></a>      age <span class="sc">+</span> sex <span class="sc">+</span> hypertension <span class="sc">+</span> hypercholesterolemia <span class="sc">+</span> hypertrigliceridemia <span class="sc">+</span></span>
<span id="cb58-8"><a href="#cb58-8" aria-hidden="true" tabindex="-1"></a>      drugs_w_loss <span class="sc">+</span> drugs_w_gain <span class="sc">+</span> bmi <span class="sc">+</span> mean_bp_mean <span class="sc">+</span> handgrip <span class="sc">+</span> evs_score <span class="sc">+</span> </span>
<span id="cb58-9"><a href="#cb58-9" aria-hidden="true" tabindex="-1"></a>      alcohol_dose <span class="sc">+</span> kcal <span class="sc">+</span> labs_alt <span class="sc">+</span> labs_ggt <span class="sc">+</span> labs_ldl <span class="sc">+</span> labs_triglycerides <span class="sc">+</span> </span>
<span id="cb58-10"><a href="#cb58-10" aria-hidden="true" tabindex="-1"></a>      labs_hdl <span class="sc">+</span> labs_quick_index,</span>
<span id="cb58-11"><a href="#cb58-11" aria-hidden="true" tabindex="-1"></a>  <span class="at">data =</span> data_crp,</span>
<span id="cb58-12"><a href="#cb58-12" aria-hidden="true" tabindex="-1"></a>  <span class="at">family =</span> <span class="fu">Gamma</span>(<span class="at">link =</span> <span class="st">"log"</span>)</span>
<span id="cb58-13"><a href="#cb58-13" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb58-14"><a href="#cb58-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb58-15"><a href="#cb58-15" aria-hidden="true" tabindex="-1"></a><span class="fu">summary</span>(pcr_glmm)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code> Family: Gamma  ( log )
Formula:          
crp_raw ~ allocation_group + visit + duration_difference + (1 |  
    record_id) + age + sex + hypertension + hypercholesterolemia +  
    hypertrigliceridemia + drugs_w_loss + drugs_w_gain + bmi +  
    mean_bp_mean + handgrip + evs_score + alcohol_dose + kcal +  
    labs_alt + labs_ggt + labs_ldl + labs_triglycerides + labs_hdl +  
    labs_quick_index
Data: data_crp

      AIC       BIC    logLik -2*log(L)  df.resid 
    636.7     704.9    -293.3     586.7        88 

Random effects:

Conditional model:
 Groups    Name        Variance Std.Dev.
 record_id (Intercept) 0.2912   0.5396  
Number of obs: 113, groups:  record_id, 73

Dispersion estimate for Gamma family (sigma^2): 0.381 

Conditional model:
                          Estimate Std. Error z value Pr(&gt;|z|)    
(Intercept)               4.807306   1.564061   3.074 0.002115 ** 
allocation_groupGrupo B   0.173671   0.209432   0.829 0.406963    
visit                     0.037019   0.091810   0.403 0.686795    
duration_difference      -0.419798   0.121190  -3.464 0.000532 ***
age                      -0.012023   0.117704  -0.102 0.918640    
sexMasculino             -0.510331   0.470362  -1.085 0.277932    
hypertension1            -0.001809   0.278499  -0.006 0.994817    
hypercholesterolemia1     0.118209   0.262977   0.450 0.653068    
hypertrigliceridemia1     0.250950   0.271486   0.924 0.355301    
drugs_w_loss1             0.284462   0.233657   1.217 0.223440    
drugs_w_gain1             0.337072   0.492272   0.685 0.493516    
bmi                       0.170129   0.107023   1.590 0.111915    
mean_bp_mean             -0.102750   0.105888  -0.970 0.331866    
handgrip                 -0.062405   0.140605  -0.444 0.657161    
evs_score                 0.232592   0.093063   2.499 0.012444 *  
alcohol_dose             -0.004767   0.088928  -0.054 0.957248    
kcal                      0.118059   0.091993   1.283 0.199371    
labs_alt                 -0.061539   0.096210  -0.640 0.522410    
labs_ggt                  0.042081   0.104095   0.404 0.686025    
labs_ldl                  0.167459   0.122997   1.361 0.173361    
labs_triglycerides       -0.143830   0.116266  -1.237 0.216059    
labs_hdl                  0.016316   0.100222   0.163 0.870675    
labs_quick_index        -11.245632   4.490688  -2.504 0.012272 *  
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1</code></pre>
</div>
<div class="sourceCode cell-code" id="cb60"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb60-1"><a href="#cb60-1" aria-hidden="true" tabindex="-1"></a><span class="fu">check_collinearity</span>(pcr_glmm)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code># Check for Multicollinearity

Low Correlation

                 Term  VIF   VIF 95% CI Increased SE Tolerance Tolerance 95% CI
     allocation_group 1.38 [1.20, 1.73]         1.17      0.72     [0.58, 0.84]
                visit 1.85 [1.54, 2.34]         1.36      0.54     [0.43, 0.65]
  duration_difference 1.60 [1.36, 2.01]         1.27      0.62     [0.50, 0.74]
                  age 1.74 [1.46, 2.19]         1.32      0.57     [0.46, 0.68]
                  sex 3.26 [2.60, 4.20]         1.81      0.31     [0.24, 0.38]
         hypertension 1.84 [1.53, 2.32]         1.36      0.54     [0.43, 0.65]
 hypercholesterolemia 1.89 [1.58, 2.39]         1.38      0.53     [0.42, 0.63]
 hypertrigliceridemia 2.31 [1.89, 2.94]         1.52      0.43     [0.34, 0.53]
         drugs_w_loss 1.29 [1.13, 1.62]         1.13      0.78     [0.62, 0.88]
         drugs_w_gain 1.18 [1.06, 1.53]         1.09      0.85     [0.66, 0.94]
                  bmi 1.22 [1.09, 1.56]         1.11      0.82     [0.64, 0.92]
         mean_bp_mean 1.64 [1.38, 2.06]         1.28      0.61     [0.49, 0.72]
             handgrip 2.51 [2.03, 3.20]         1.58      0.40     [0.31, 0.49]
            evs_score 1.40 [1.21, 1.76]         1.18      0.71     [0.57, 0.82]
         alcohol_dose 1.63 [1.38, 2.05]         1.28      0.61     [0.49, 0.73]
                 kcal 1.52 [1.30, 1.90]         1.23      0.66     [0.53, 0.77]
             labs_alt 1.47 [1.26, 1.85]         1.21      0.68     [0.54, 0.79]
             labs_ggt 1.66 [1.40, 2.08]         1.29      0.60     [0.48, 0.72]
             labs_ldl 2.03 [1.68, 2.57]         1.43      0.49     [0.39, 0.60]
   labs_triglycerides 2.24 [1.84, 2.85]         1.50      0.45     [0.35, 0.54]
             labs_hdl 1.33 [1.16, 1.67]         1.15      0.75     [0.60, 0.86]
     labs_quick_index 1.57 [1.34, 1.97]         1.25      0.64     [0.51, 0.75]</code></pre>
</div>
<div class="sourceCode cell-code" id="cb62"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb62-1"><a href="#cb62-1" aria-hidden="true" tabindex="-1"></a><span class="fu">AIC</span>(pcr_glmm)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>[1] 636.6772</code></pre>
</div>
<div class="sourceCode cell-code" id="cb64"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb64-1"><a href="#cb64-1" aria-hidden="true" tabindex="-1"></a><span class="fu">BIC</span>(pcr_glmm)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>[1] 704.8619</code></pre>
</div>
<div class="sourceCode cell-code" id="cb66"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb66-1"><a href="#cb66-1" aria-hidden="true" tabindex="-1"></a><span class="fu">r2</span>(pcr_glmm)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code># R2 for Mixed Models

  Conditional R2: 0.676
     Marginal R2: 0.385</code></pre>
</div>
<div class="sourceCode cell-code" id="cb68"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb68-1"><a href="#cb68-1" aria-hidden="true" tabindex="-1"></a>pcr_glmm_plots <span class="ot">&lt;-</span> performance<span class="sc">::</span><span class="fu">check_model</span>(pcr_1)</span>
<span id="cb68-2"><a href="#cb68-2" aria-hidden="true" tabindex="-1"></a><span class="fu">print</span>(pcr_glmm_plots)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="Results_main_outcomes_files/figure-html/unnamed-chunk-14-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
<p><strong>Full Analysis of the GLMM (Gamma, Log-Link)</strong></p>
<p><em>Model Fit and Diagnostics</em></p>
<ul>
<li>AIC: 636.7</li>
<li>BIC: 704.9</li>
<li>Marginal R²: 0.385 (variance explained by fixed effects)</li>
<li>Conditional R²: 0.676 (variance explained by fixed + random effects)</li>
</ul>
<p><em>Interpretation:</em></p>
<ul>
<li>This represents a substantial improvement in explained variance relative to the linear mixed model (marginal R² ~0.29).</li>
<li>The higher conditional R² suggests that between-subject variability (random intercept) remains important.</li>
</ul>
<p><em>Diagnostics:</em> - Posterior Predictive Check: Good alignment between model-predicted density and observed data. No major distributional misspecification. - Linearity: Acceptable. Slight deviations, but no systematic bias. - Homoscedasticity: Vastly improved relative to the LMM. The variance is well-behaved along the fitted line, which is expected for a Gamma model. - Normality of Residuals: Strong. No heavy tails or skewness issues. - Influential Observations: Leverage and residual plots are within safe boundaries. No problematic points. - Collinearity: VIFs are all below 3.3, no collinearity concerns.</p>
<p>Key Fixed Effects</p>
<table class="caption-top table">
<colgroup>
<col style="width: 25%">
<col style="width: 10%">
<col style="width: 10%">
<col style="width: 52%">
</colgroup>
<thead>
<tr class="header">
<th>Variable</th>
<th>Estimate</th>
<th>p-value</th>
<th>Interpretation</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Intercept</td>
<td>4.807</td>
<td>0.002</td>
<td>Baseline CRP level (on log scale)</td>
</tr>
<tr class="even">
<td>Duration Difference</td>
<td>-0.420</td>
<td>0.0005</td>
<td>Significant. Each 1 SD longer than planned duration leads to a 34% reduction in CRP (exp(-0.42) ≈ 0.66)</td>
</tr>
<tr class="odd">
<td>EVS Score</td>
<td>0.233</td>
<td>0.012</td>
<td>Significant. Higher physical activity is associated with 26% higher CRP, which is counterintuitive and requires biological scrutiny. May reflect confounding.</td>
</tr>
<tr class="even">
<td>Labs QUICKI</td>
<td>-11.246</td>
<td>0.012</td>
<td>Strongly significant. Higher insulin sensitivity leads to &gt;99% reduction in CRP, suggesting a very steep relationship in the data. Likely influenced by scaling.</td>
</tr>
<tr class="odd">
<td>Allocation Group</td>
<td>0.174</td>
<td>0.407</td>
<td>Not significant. Consistent with a per-protocol analytical framework.</td>
</tr>
<tr class="even">
<td>Visit</td>
<td>0.037</td>
<td>0.687</td>
<td>Not significant. No residual time effect beyond the modeled variables.</td>
</tr>
<tr class="odd">
<td>BMI</td>
<td>0.170</td>
<td>0.112</td>
<td>Borderline. Each SD increase in BMI tends to raise CRP by ~18%, but not significant at p&lt;0.05.</td>
</tr>
</tbody>
</table>
<p>All other clinical covariates (age, sex, hypertension, lipid markers, liver enzymes) were non-significant, indicating that the major drivers in this dataset are related to treatment adherence (duration), metabolic health (QUICKI), and physical activity (EVS).</p>
<p>Biological Plausibility</p>
<ul>
<li>Duration Difference Negative Effect: Completely plausible. Lower deviation from planned duration (i.e., better adherence) leads to reduced inflammation, as expected.</li>
<li>QUICKI Negative Effect: Consistent with insulin resistance being a pro-inflammatory state. The very large coefficient likely reflects the scale of QUICKI (small numeric values like ~0.3–0.4).</li>
<li>EVS Positive Effect: Counterintuitive. Suggests participants with higher reported physical activity had higher CRP. This could be:
<ul>
<li>A measurement error in EVS (self-reported bias).</li>
<li>Confounding by illness severity (e.g., those with higher baseline CRP trying to increase activity).</li>
<li>An artifact of the small sample size.</li>
<li>Worth sensitivity testing.</li>
</ul></li>
</ul>
<p>Model Performance</p>
<ul>
<li>This GLMM offers superior handling of heteroscedasticity compared to the LMM.</li>
<li>The residual plots confirm that variance increases with the mean, which the Gamma distribution models correctly.</li>
<li>The gain in marginal R² from ~0.29 (LMM) to 0.385 (GLMM) is meaningful.</li>
<li>Model is stable with no multicollinearity, good residual behavior, and sound random effects distribution.</li>
</ul>
<p>Conclusion</p>
<ul>
<li>This GLMM is statistically robust and biologically meaningful.</li>
<li>The key drivers of CRP in this study are:
<ul>
<li>Adherence (duration difference).</li>
<li>Insulin sensitivity (QUICKI).</li>
<li>Physical activity (EVS), with a result that warrants further scrutiny.</li>
</ul></li>
</ul>
<p>Recommendation</p>
<ul>
<li>Yes, this model should be your primary model for reporting.</li>
<li>Report exponentiated coefficients for clinical interpretability.</li>
<li>Consider sensitivity analyses:
<ul>
<li>Check whether removing outliers or modifying EVS affects results.</li>
<li>Test a simplified model without EVS to observe stability.</li>
</ul></li>
<li>Prepare Results and Discussion sections reflecting that GLMM offered a better statistical fit than LMM due to the appropriate modeling of the CRP distribution.</li>
</ul>
<p>If you want, I can help you draft the precise sentences for the manuscript results and discussion.</p>
</section>
<section id="tweedie" class="level4">
<h4 class="anchored" data-anchor-id="tweedie">Tweedie</h4>
<div class="cell">
<div class="sourceCode cell-code" id="cb69"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb69-1"><a href="#cb69-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Load required libraries</span></span>
<span id="cb69-2"><a href="#cb69-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(glmmTMB)</span>
<span id="cb69-3"><a href="#cb69-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(performance)</span>
<span id="cb69-4"><a href="#cb69-4" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ggeffects)</span>
<span id="cb69-5"><a href="#cb69-5" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(DHARMa)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>This is DHARMa 0.4.7. For overview type '?DHARMa'. For recent changes, type news(package = 'DHARMa')</code></pre>
</div>
<div class="sourceCode cell-code" id="cb71"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb71-1"><a href="#cb71-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Fit the Tweedie GLMM</span></span>
<span id="cb71-2"><a href="#cb71-2" aria-hidden="true" tabindex="-1"></a>pcr_tweedie <span class="ot">&lt;-</span> <span class="fu">glmmTMB</span>(</span>
<span id="cb71-3"><a href="#cb71-3" aria-hidden="true" tabindex="-1"></a>  crp_raw <span class="sc">~</span> </span>
<span id="cb71-4"><a href="#cb71-4" aria-hidden="true" tabindex="-1"></a>    allocation_group <span class="sc">+</span> visit <span class="sc">+</span> duration_difference <span class="sc">+</span> </span>
<span id="cb71-5"><a href="#cb71-5" aria-hidden="true" tabindex="-1"></a>    (<span class="dv">1</span> <span class="sc">|</span> record_id) <span class="sc">+</span> </span>
<span id="cb71-6"><a href="#cb71-6" aria-hidden="true" tabindex="-1"></a>    age <span class="sc">+</span> sex <span class="sc">+</span> hypertension <span class="sc">+</span> hypercholesterolemia <span class="sc">+</span> hypertrigliceridemia <span class="sc">+</span> </span>
<span id="cb71-7"><a href="#cb71-7" aria-hidden="true" tabindex="-1"></a>    drugs_w_loss <span class="sc">+</span> drugs_w_gain <span class="sc">+</span> bmi <span class="sc">+</span> mean_bp_mean <span class="sc">+</span> handgrip <span class="sc">+</span> evs_score <span class="sc">+</span> </span>
<span id="cb71-8"><a href="#cb71-8" aria-hidden="true" tabindex="-1"></a>    alcohol_dose <span class="sc">+</span> kcal <span class="sc">+</span> labs_alt <span class="sc">+</span> labs_ggt <span class="sc">+</span> labs_ldl <span class="sc">+</span> labs_triglycerides <span class="sc">+</span> </span>
<span id="cb71-9"><a href="#cb71-9" aria-hidden="true" tabindex="-1"></a>    labs_hdl <span class="sc">+</span> labs_quick_index,</span>
<span id="cb71-10"><a href="#cb71-10" aria-hidden="true" tabindex="-1"></a>  <span class="at">data =</span> data_crp,</span>
<span id="cb71-11"><a href="#cb71-11" aria-hidden="true" tabindex="-1"></a>  <span class="at">family =</span> <span class="fu">tweedie</span>(<span class="at">link =</span> <span class="st">"log"</span>)</span>
<span id="cb71-12"><a href="#cb71-12" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb71-13"><a href="#cb71-13" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb71-14"><a href="#cb71-14" aria-hidden="true" tabindex="-1"></a><span class="co"># Summary of the model</span></span>
<span id="cb71-15"><a href="#cb71-15" aria-hidden="true" tabindex="-1"></a><span class="fu">summary</span>(pcr_tweedie)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code> Family: tweedie  ( log )
Formula:          
crp_raw ~ allocation_group + visit + duration_difference + (1 |  
    record_id) + age + sex + hypertension + hypercholesterolemia +  
    hypertrigliceridemia + drugs_w_loss + drugs_w_gain + bmi +  
    mean_bp_mean + handgrip + evs_score + alcohol_dose + kcal +  
    labs_alt + labs_ggt + labs_ldl + labs_triglycerides + labs_hdl +  
    labs_quick_index
Data: data_crp

      AIC       BIC    logLik -2*log(L)  df.resid 
    638.3     709.2    -293.2     586.3        87 

Random effects:

Conditional model:
 Groups    Name        Variance Std.Dev.
 record_id (Intercept) 0.3088   0.5557  
Number of obs: 113, groups:  record_id, 73

Dispersion parameter for tweedie family (): 0.417 

Conditional model:
                          Estimate Std. Error z value Pr(&gt;|z|)    
(Intercept)               4.564286   1.544326   2.956 0.003121 ** 
allocation_groupGrupo B   0.155117   0.209424   0.741 0.458886    
visit                     0.030017   0.088831   0.338 0.735427    
duration_difference      -0.413774   0.120599  -3.431 0.000601 ***
age                      -0.014737   0.115579  -0.128 0.898541    
sexMasculino             -0.503895   0.464336  -1.085 0.277835    
hypertension1            -0.007849   0.275488  -0.028 0.977269    
hypercholesterolemia1     0.105401   0.260406   0.405 0.685656    
hypertrigliceridemia1     0.259246   0.266598   0.972 0.330839    
drugs_w_loss1             0.283488   0.229910   1.233 0.217561    
drugs_w_gain1             0.320096   0.485889   0.659 0.510035    
bmi                       0.168685   0.105322   1.602 0.109240    
mean_bp_mean             -0.096418   0.104162  -0.926 0.354622    
handgrip                 -0.059524   0.138256  -0.431 0.666804    
evs_score                 0.222506   0.092318   2.410 0.015943 *  
alcohol_dose             -0.008930   0.086312  -0.103 0.917595    
kcal                      0.109883   0.091999   1.194 0.232325    
labs_alt                 -0.061716   0.095036  -0.649 0.516087    
labs_ggt                  0.037796   0.105847   0.357 0.721032    
labs_ldl                  0.165804   0.119591   1.386 0.165618    
labs_triglycerides       -0.134222   0.111598  -1.203 0.229082    
labs_hdl                  0.015942   0.098417   0.162 0.871316    
labs_quick_index        -10.450474   4.451319  -2.348 0.018888 *  
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1</code></pre>
</div>
<div class="sourceCode cell-code" id="cb73"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb73-1"><a href="#cb73-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Check collinearity</span></span>
<span id="cb73-2"><a href="#cb73-2" aria-hidden="true" tabindex="-1"></a><span class="fu">check_collinearity</span>(pcr_tweedie)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code># Check for Multicollinearity

Low Correlation

                 Term  VIF   VIF 95% CI Increased SE Tolerance Tolerance 95% CI
     allocation_group 1.39 [1.20, 1.74]         1.18      0.72     [0.57, 0.83]
                visit 1.88 [1.57, 2.37]         1.37      0.53     [0.42, 0.64]
  duration_difference 1.59 [1.35, 2.00]         1.26      0.63     [0.50, 0.74]
                  age 1.71 [1.44, 2.16]         1.31      0.58     [0.46, 0.69]
                  sex 3.11 [2.48, 3.99]         1.76      0.32     [0.25, 0.40]
         hypertension 1.80 [1.51, 2.27]         1.34      0.55     [0.44, 0.66]
 hypercholesterolemia 1.87 [1.55, 2.36]         1.37      0.54     [0.42, 0.64]
 hypertrigliceridemia 2.24 [1.83, 2.85]         1.50      0.45     [0.35, 0.55]
         drugs_w_loss 1.27 [1.12, 1.61]         1.13      0.79     [0.62, 0.89]
         drugs_w_gain 1.19 [1.07, 1.53]         1.09      0.84     [0.65, 0.94]
                  bmi 1.22 [1.09, 1.55]         1.10      0.82     [0.64, 0.92]
         mean_bp_mean 1.64 [1.39, 2.06]         1.28      0.61     [0.49, 0.72]
             handgrip 2.42 [1.97, 3.08]         1.56      0.41     [0.32, 0.51]
            evs_score 1.42 [1.23, 1.78]         1.19      0.70     [0.56, 0.81]
         alcohol_dose 1.60 [1.36, 2.01]         1.26      0.63     [0.50, 0.74]
                 kcal 1.55 [1.32, 1.94]         1.24      0.65     [0.52, 0.76]
             labs_alt 1.47 [1.26, 1.84]         1.21      0.68     [0.54, 0.79]
             labs_ggt 1.63 [1.38, 2.05]         1.28      0.61     [0.49, 0.72]
             labs_ldl 2.01 [1.66, 2.54]         1.42      0.50     [0.39, 0.60]
   labs_triglycerides 2.16 [1.78, 2.74]         1.47      0.46     [0.36, 0.56]
             labs_hdl 1.32 [1.16, 1.66]         1.15      0.76     [0.60, 0.87]
     labs_quick_index 1.56 [1.32, 1.95]         1.25      0.64     [0.51, 0.75]</code></pre>
</div>
<div class="sourceCode cell-code" id="cb75"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb75-1"><a href="#cb75-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Model fit indices</span></span>
<span id="cb75-2"><a href="#cb75-2" aria-hidden="true" tabindex="-1"></a><span class="fu">AIC</span>(pcr_tweedie)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>[1] 638.3089</code></pre>
</div>
<div class="sourceCode cell-code" id="cb77"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb77-1"><a href="#cb77-1" aria-hidden="true" tabindex="-1"></a><span class="fu">BIC</span>(pcr_tweedie)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>[1] 709.221</code></pre>
</div>
<div class="sourceCode cell-code" id="cb79"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb79-1"><a href="#cb79-1" aria-hidden="true" tabindex="-1"></a><span class="co"># R-squared</span></span>
<span id="cb79-2"><a href="#cb79-2" aria-hidden="true" tabindex="-1"></a><span class="fu">r2</span>(pcr_tweedie)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code># R2 for Mixed Models

  Conditional R2: 0.698
     Marginal R2: 0.379</code></pre>
</div>
<div class="sourceCode cell-code" id="cb81"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb81-1"><a href="#cb81-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Residual diagnostics with DHARMa</span></span>
<span id="cb81-2"><a href="#cb81-2" aria-hidden="true" tabindex="-1"></a>simulation_output <span class="ot">&lt;-</span> <span class="fu">simulateResiduals</span>(<span class="at">fittedModel =</span> pcr_tweedie)</span>
<span id="cb81-3"><a href="#cb81-3" aria-hidden="true" tabindex="-1"></a><span class="fu">plot</span>(simulation_output)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="Results_main_outcomes_files/figure-html/unnamed-chunk-15-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="sourceCode cell-code" id="cb82"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb82-1"><a href="#cb82-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Model diagnostics with performance</span></span>
<span id="cb82-2"><a href="#cb82-2" aria-hidden="true" tabindex="-1"></a><span class="fu">check_model</span>(pcr_tweedie)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>`check_outliers()` does not yet support models of class `glmmTMB`.</code></pre>
</div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="Results_main_outcomes_files/figure-html/unnamed-chunk-15-2.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="sourceCode cell-code" id="cb84"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb84-1"><a href="#cb84-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Plot predicted effects for key variables (example: allocation group)</span></span>
<span id="cb84-2"><a href="#cb84-2" aria-hidden="true" tabindex="-1"></a>ggeffects<span class="sc">::</span><span class="fu">ggpredict</span>(pcr_tweedie, <span class="at">terms =</span> <span class="st">"allocation_group"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb84-3"><a href="#cb84-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">plot</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="Results_main_outcomes_files/figure-html/unnamed-chunk-15-3.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="sourceCode cell-code" id="cb85"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb85-1"><a href="#cb85-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Plot for a continuous variable (example: evs_score)</span></span>
<span id="cb85-2"><a href="#cb85-2" aria-hidden="true" tabindex="-1"></a>ggeffects<span class="sc">::</span><span class="fu">ggpredict</span>(pcr_tweedie, <span class="at">terms =</span> <span class="st">"evs_score [all]"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb85-3"><a href="#cb85-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">plot</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="Results_main_outcomes_files/figure-html/unnamed-chunk-15-4.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
<p>Is the Tweedie better than the simple GLMM?</p>
<ol type="1">
<li>Model Fit Comparison</li>
</ol>
<table class="caption-top table">
<thead>
<tr class="header">
<th>Model</th>
<th>AIC</th>
<th>BIC</th>
<th>Conditional R²</th>
<th>Marginal R²</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>GLMM (Gamma)</td>
<td>636.7</td>
<td>704.9</td>
<td>0.676</td>
<td>0.385</td>
</tr>
<tr class="even">
<td>GLMM (Tweedie)</td>
<td>638.3</td>
<td>709.2</td>
<td>0.698</td>
<td>0.379</td>
</tr>
</tbody>
</table>
<ul>
<li>The AIC and BIC are slightly better for the Gamma model, but the difference is negligible (&lt;2 points).</li>
<li>The Tweedie has a slightly higher Conditional R², meaning it captures slightly more total variance.</li>
<li>Marginal R² is essentially the same.</li>
</ul>
<ol start="2" type="1">
<li>Residual Diagnostics</li>
</ol>
<table class="caption-top table">
<thead>
<tr class="header">
<th>Diagnostic Aspect</th>
<th>GLMM (Gamma)</th>
<th>GLMM (Tweedie)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Homogeneity</td>
<td>Some issue</td>
<td>Some issue</td>
</tr>
<tr class="even">
<td>Normality of residuals</td>
<td>Acceptable</td>
<td>Slightly better</td>
</tr>
<tr class="odd">
<td>DHARMa tests</td>
<td>Passed</td>
<td>Passed</td>
</tr>
</tbody>
</table>
<ul>
<li>Both models show mild heteroscedasticity.</li>
<li>Tweedie has slightly better QQ residual behavior.</li>
<li>Both models pass DHARMa diagnostics.</li>
</ul>
<ol start="3" type="1">
<li>Theoretical Suitability</li>
</ol>
<ul>
<li>Gamma assumes positive continuous data with variance proportional to the mean squared.</li>
<li>Tweedie is more flexible, with variance proportional to mean^p (p between 1 and 2 for continuous positive skewed data).</li>
<li>Tweedie generalizes Gamma and is conceptually better for heavily right-skewed biomedical data like CRP.</li>
</ul>
<ol start="4" type="1">
<li>Practical Implications</li>
</ol>
<ul>
<li>The estimates of fixed effects are very similar between models.</li>
<li>The same predictors are statistically significant in both.</li>
</ul>
<ol start="5" type="1">
<li>Conclusion</li>
</ol>
<ul>
<li>Statistically: Not clearly better. Equivalent fit with negligible differences.</li>
<li>Conceptually: Tweedie is preferable because it has a more flexible variance structure suitable for this kind of data.</li>
<li>Recommendation: Tweedie is more defensible for publication. Both models yield consistent conclusions, which reinforces the robustness of the results.</li>
</ul>
<p>Parameter Estimates Comparison</p>
<table class="caption-top table">
<colgroup>
<col style="width: 28%">
<col style="width: 19%">
<col style="width: 21%">
<col style="width: 14%">
<col style="width: 16%">
</colgroup>
<thead>
<tr class="header">
<th>Predictor</th>
<th>Gamma Estimate</th>
<th>Tweedie Estimate</th>
<th>Direction</th>
<th>Significance</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>(Intercept)</td>
<td>4.807</td>
<td>4.564</td>
<td>Positive</td>
<td>**</td>
</tr>
<tr class="even">
<td>allocation_groupGrupoB</td>
<td>0.174</td>
<td>0.155</td>
<td>Positive</td>
<td>ns</td>
</tr>
<tr class="odd">
<td>visit</td>
<td>0.037</td>
<td>0.030</td>
<td>Positive</td>
<td>ns</td>
</tr>
<tr class="even">
<td>duration_difference</td>
<td>-0.420</td>
<td>-0.414</td>
<td>Negative</td>
<td>***</td>
</tr>
<tr class="odd">
<td>age</td>
<td>-0.012</td>
<td>-0.015</td>
<td>Negative</td>
<td>ns</td>
</tr>
<tr class="even">
<td>sexMasculino</td>
<td>-0.510</td>
<td>-0.504</td>
<td>Negative</td>
<td>ns</td>
</tr>
<tr class="odd">
<td>hypertension1</td>
<td>-0.002</td>
<td>-0.008</td>
<td>Negative</td>
<td>ns</td>
</tr>
<tr class="even">
<td>hypercholesterolemia1</td>
<td>0.118</td>
<td>0.105</td>
<td>Positive</td>
<td>ns</td>
</tr>
<tr class="odd">
<td>hypertrigliceridemia1</td>
<td>0.251</td>
<td>0.259</td>
<td>Positive</td>
<td>ns</td>
</tr>
<tr class="even">
<td>drugs_w_loss1</td>
<td>0.284</td>
<td>0.283</td>
<td>Positive</td>
<td>ns</td>
</tr>
<tr class="odd">
<td>drugs_w_gain1</td>
<td>0.337</td>
<td>0.320</td>
<td>Positive</td>
<td>ns</td>
</tr>
<tr class="even">
<td>bmi</td>
<td>0.170</td>
<td>0.169</td>
<td>Positive</td>
<td>. (trend)</td>
</tr>
<tr class="odd">
<td>mean_bp_mean</td>
<td>-0.103</td>
<td>-0.096</td>
<td>Negative</td>
<td>ns</td>
</tr>
<tr class="even">
<td>handgrip</td>
<td>-0.062</td>
<td>-0.060</td>
<td>Negative</td>
<td>ns</td>
</tr>
<tr class="odd">
<td>evs_score</td>
<td>0.233</td>
<td>0.223</td>
<td>Positive</td>
<td>*</td>
</tr>
<tr class="even">
<td>alcohol_dose</td>
<td>-0.005</td>
<td>-0.009</td>
<td>Negative</td>
<td>ns</td>
</tr>
<tr class="odd">
<td>kcal</td>
<td>0.118</td>
<td>0.110</td>
<td>Positive</td>
<td>ns</td>
</tr>
<tr class="even">
<td>labs_alt</td>
<td>-0.062</td>
<td>-0.062</td>
<td>Negative</td>
<td>ns</td>
</tr>
<tr class="odd">
<td>labs_ggt</td>
<td>0.042</td>
<td>0.038</td>
<td>Positive</td>
<td>ns</td>
</tr>
<tr class="even">
<td>labs_ldl</td>
<td>0.167</td>
<td>0.166</td>
<td>Positive</td>
<td>ns</td>
</tr>
<tr class="odd">
<td>labs_triglycerides</td>
<td>-0.144</td>
<td>-0.134</td>
<td>Negative</td>
<td>ns</td>
</tr>
<tr class="even">
<td>labs_hdl</td>
<td>0.016</td>
<td>0.016</td>
<td>Positive</td>
<td>ns</td>
</tr>
<tr class="odd">
<td>labs_quick_index</td>
<td>-11.246</td>
<td>-10.450</td>
<td>Negative</td>
<td>*</td>
</tr>
</tbody>
</table>
<ul>
<li>Direction of effect is consistent across both models.</li>
<li>Significance patterns are identical.</li>
</ul>
<p>Summary: Tweedie offers slightly better conceptual fit without compromising statistical robustness. Both models are valid and lead to the same scientific conclusions.</p>
</section>
<section id="other-models" class="level4">
<h4 class="anchored" data-anchor-id="other-models">Other models</h4>
<ol type="1">
<li><p>Linear Mixed Model on ΔCRP (Change Score). Collapse the data to one row per participant and model the change in CRP from baseline to final visit. This often reduces heteroscedasticity since we’re modeling change rather than absolute levels.</p></li>
<li><p>Quantile Mixed Models. Instead of modeling the mean, quantile models can model medians or other quantiles, which are robust to heteroscedasticity and skewness. Example package: lqmm. Models the median instead of the mean — inherently robust to heteroscedasticity.</p></li>
<li><p>Robust Mixed Models. Mixed models with robust estimation of residual variance, less sensitive to heteroscedasticity or outliers. Same formula as LMM, but with robust handling of residual variance.</p></li>
<li><p>Model the Raw CRP with a Two-Part Model (If Many Zeros). If there are many zeros or very low values, a two-part hurdle model or zero-inflated model may better capture the data structure. This is less common for CRP unless a substantial portion is zero or below detection limit.</p></li>
</ol>
<p>Recommendation hierarchy: 1. First choice: → Try Gamma GLMM with log link using glmmTMB — best suited for continuous, right-skewed, strictly positive outcomes like CRP. 2. If interest is in change rather than absolute levels: → Model ΔCRP (change score) with LMM. 3. If robustness to outliers is crucial: → Try robust mixed models or quantile mixed models.</p>
</section>
</section>
</section>
<section id="modeling-compliance" class="level2">
<h2 class="anchored" data-anchor-id="modeling-compliance">Modeling compliance</h2>
<p>Since the goal is to evaluate whether the intervention (allocation_group) reduces inflammation (log1p(labs_crp)), then how compliance-related variables are handled — completed_intervention, compliance_score_adjusted, and duration_difference — fundamentally changes what question the model is answering.</p>
<p><strong>Key Conceptual Framework</strong></p>
<p>If we adjust for compliance variables, then we are answering the question: Among those who complied (to varying degrees), does the assigned intervention have an effect on inflammation?”. This estimates the “Effect of Treatment on the Treated” (ETT) or per-protocol effect, rather than the intention-to-treat (ITT) effect. The interpretation shifts towards whether the efficacy of the treatment holds after accounting for how well participants adhered. If participants in the intervention group did not comply, the model adjusts for that, isolating the treatment effect conditional on compliance. However, if compliance is strongly influenced by the intervention (as is common), adjusting for it may introduce <strong>post-treatment bias (collider bias)</strong>, because we’re conditioning on a variable that is downstream of the allocation.</p>
<p>If we do not adjust for compliance, then we are estimating the intention-to-treat (ITT) effect: “Does being assigned to the intervention reduce inflammation, regardless of whether participants fully complied?”. This reflects the real-world effectiveness including factors like dropout, low adherence, etc. This is the primary approach in most randomized trials because it preserves the randomization balance and avoids bias introduced by post-treatment variables.</p>
<p>In the current model, we are conditioning the effect of allocation_group on adherence. The coefficient for allocation_group in this case answers: “Given the same level of adherence (completion, dose, duration), is the intervention effective in reducing inflammation?” In other words, we’re stripping out the pathway by which poor adherence could dilute the intervention effect. If these variables are strong mediators of the intervention’s effect, we are blocking part of the effect of allocation_group, and this might explain why allocation_group is not significant.</p>
<p><strong>Correct Modeling Strategy</strong></p>
<p>Depends on our goal:</p>
<ol type="1">
<li><p>To estimate the pure biological effect of the intervention (conditional on adherence):</p>
<ul>
<li>Keep compliance_score_adjusted, completed_intervention, and duration_difference in the model.</li>
<li>Interpret the effect of allocation_group as the effect assuming equal adherence across groups.</li>
</ul></li>
<li><p>To estimate the real-world effectiveness (intention-to-treat):</p>
<ul>
<li>Remove the compliance-related variables from the model.</li>
<li>The coefficient for allocation_group will capture the total effect, including any dilution due to poor adherence.</li>
</ul></li>
</ol>
<p><img src="images/DAG:%20Relationship%20between%20Intervention,%20Compliance,%20and%20Inflammation.png" class="img-fluid"></p>
<p>The DAG (Directed Acyclic Graph) represents the causal relationships between the intervention, adherence, and inflammation. In this structure, Allocation Group has a direct effect on Inflammation (labs_crp) and also influences Compliance, Completed Intervention, and Duration Difference, which are measures of adherence. These adherence-related variables, in turn, affect Inflammation, indicating that part of the intervention’s effect is mediated through adherence. Additionally, a hypothetical Unmeasured Factors node, representing elements such as motivation, health status, or life circumstances, influences both Compliance and Inflammation. This creates a potential collider if the analysis conditions on adherence-related variables.</p>
<p>The DAG highlights that adjusting for compliance, completion, or duration blocks part of the natural causal pathway through which the intervention affects inflammation. Moreover, conditioning on these variables opens a backdoor path through Unmeasured Factors, introducing potential collider bias.</p>
<p>This structure demonstrates that the choice between intention-to-treat (ITT) and per-protocol (PP) analysis changes the research question. In ITT analysis, the effect of the Allocation Group on inflammation is estimated without conditioning on adherence, capturing the total effect, including both direct and adherence-mediated pathways. This approach preserves the benefits of randomization and estimates the real-world effectiveness of the intervention. In contrast, per-protocol analysis includes adherence-related variables in the model. This blocks the indirect pathways through adherence and estimates the effect of the intervention among participants with comparable adherence. While this approach can answer questions about biological efficacy conditional on adherence, it carries a higher risk of bias due to conditioning on post-randomization variables.</p>
<p><strong>Suggested Dual Approach (Best Practice in Clinical Research)</strong></p>
<ol type="1">
<li>Primary Model (ITT) without compliance variables. This estimates whether the offer of the intervention reduces inflammation.</li>
</ol>
<pre><code>log1p(labs_crp) \~ allocation_group + visit + (1 \| record_id) + covariates</code></pre>
<ol start="2" type="1">
<li>Secondary Model (Per-Protocol Sensitivity Analysis) with compliance variables. This estimates whether the intervention works among those who actually adhered to it, or to account for variability in exposure.</li>
</ol>
<pre><code>log1p(labs_crp) \~ allocation_group + compliance_score_adjusted + duration_difference + completed_intervention + visit + (1 \| record_id) + covariates</code></pre>
<p><strong>Important Note on Mediation vs.&nbsp;Confounding:</strong></p>
<ul>
<li>Compliance is not a confounder — it occurs after randomization.</li>
<li>Conditioning on it changes the causal question.</li>
<li>Including post-randomization variables risks collider bias if those variables are influenced by both the intervention and unmeasured factors affecting inflammation.</li>
</ul>
<p><strong>Final Recommendation for Writing in Thesis:</strong></p>
<p>The primary analysis was conducted under the intention-to-treat principle, evaluating the effect of the randomized intervention on inflammation regardless of adherence. A secondary model adjusted for adherence-related variables (completed_intervention, compliance_score_adjusted, and duration_difference) to explore whether the intervention had an effect among those who maintained adequate compliance. These models capture complementary aspects of intervention effectiveness: the former reflects real-world applicability, while the latter reflects efficacy conditional on adequate adherence.</p>
<section id="ideal-condition" class="level4">
<h4 class="anchored" data-anchor-id="ideal-condition">Ideal condition</h4>
<p>The first model (pcr_1) represents a per-protocol-type model, where the effect of the intervention (allocation_group) is estimated conditional on compliance (through compliance_score_adjusted, completed_intervention, and duration_difference). However, there are other methods that can estimate the effect of the intervention in an ideal condition, where adherence is perfect. These methods go beyond simply conditioning in regression and attempt to more rigorously isolate the causal effect.</p>
<p><strong>Alternative and Complementary Strategies to Per-Protocol Modeling</strong></p>
<p><em>1. Restricting the Dataset (Complete Compliers Only)</em></p>
<p>One simple but crude way to perform a per-protocol analysis is to exclude participants who did not complete the intervention or had poor compliance. This creates a model within a population that fully adhered. The limitations are: reduces sample size and breaks randomization, leading to confounding.</p>
<pre><code>data_pp &lt;- data_model_scaled %&gt;%
  filter(completed_intervention == "Yes", compliance_score_adjusted &gt;= threshold)

pcr_pp &lt;- lmer(log1p(labs_crp) ~ allocation_group + visit + (1 | record_id) + covariates, data = data_pp)</code></pre>
<div class="cell">
<div class="sourceCode cell-code" id="cb89"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb89-1"><a href="#cb89-1" aria-hidden="true" tabindex="-1"></a>data_pp <span class="ot">&lt;-</span> data_model_scaled <span class="sc">%&gt;%</span></span>
<span id="cb89-2"><a href="#cb89-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(</span>
<span id="cb89-3"><a href="#cb89-3" aria-hidden="true" tabindex="-1"></a>    completed_intervention <span class="sc">==</span> <span class="st">"Sim"</span> <span class="sc">&amp;</span></span>
<span id="cb89-4"><a href="#cb89-4" aria-hidden="true" tabindex="-1"></a>      (compliance_score_visit <span class="sc">&gt;</span> <span class="fl">0.6</span> <span class="sc">|</span> <span class="fu">is.na</span>(compliance_score_visit))</span>
<span id="cb89-5"><a href="#cb89-5" aria-hidden="true" tabindex="-1"></a>  )</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="restrict-dataset" class="level4">
<h4 class="anchored" data-anchor-id="restrict-dataset">Restrict dataset</h4>
<div class="cell">
<div class="sourceCode cell-code" id="cb90"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb90-1"><a href="#cb90-1" aria-hidden="true" tabindex="-1"></a>pcr_pp <span class="ot">&lt;-</span> <span class="fu">lmer</span>(<span class="fu">log1p</span>(labs_crp) <span class="sc">~</span> allocation_group <span class="sc">+</span> duration_difference <span class="sc">+</span> visit <span class="sc">+</span> (<span class="dv">1</span> <span class="sc">|</span> record_id) <span class="sc">+</span> age <span class="sc">+</span> sex <span class="sc">+</span> hypertension <span class="sc">+</span> hypercholesterolemia <span class="sc">+</span> hypertrigliceridemia <span class="sc">+</span> drugs_w_loss <span class="sc">+</span> drugs_w_gain <span class="sc">+</span> mean_bp_mean <span class="sc">+</span> handgrip <span class="sc">+</span> evs_score <span class="sc">+</span> alcohol_dose <span class="sc">+</span> kcal <span class="sc">+</span> labs_alt <span class="sc">+</span> labs_ggt <span class="sc">+</span> labs_ldl <span class="sc">+</span> labs_triglycerides <span class="sc">+</span> labs_hdl <span class="sc">+</span> labs_quick_index, <span class="at">data =</span> data_pp)</span>
<span id="cb90-2"><a href="#cb90-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb90-3"><a href="#cb90-3" aria-hidden="true" tabindex="-1"></a><span class="fu">summary</span>(pcr_pp)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>Linear mixed model fit by REML. t-tests use Satterthwaite's method [
lmerModLmerTest]
Formula: log1p(labs_crp) ~ allocation_group + duration_difference + visit +  
    (1 | record_id) + age + sex + hypertension + hypercholesterolemia +  
    hypertrigliceridemia + drugs_w_loss + drugs_w_gain + mean_bp_mean +  
    handgrip + evs_score + alcohol_dose + kcal + labs_alt + labs_ggt +  
    labs_ldl + labs_triglycerides + labs_hdl + labs_quick_index
   Data: data_pp

REML criterion at convergence: 211.7

Scaled residuals: 
    Min      1Q  Median      3Q     Max 
-1.8596 -0.4588 -0.0302  0.4374  3.4390 

Random effects:
 Groups    Name        Variance Std.Dev.
 record_id (Intercept) 0.1825   0.4272  
 Residual              0.3910   0.6253  
Number of obs: 87, groups:  record_id, 52

Fixed effects:
                        Estimate Std. Error       df t value Pr(&gt;|t|)  
(Intercept)              1.30114    1.51620 59.82978   0.858   0.3942  
allocation_groupGrupo B -0.14874    0.22985 34.14011  -0.647   0.5219  
duration_difference     -0.33950    0.12820 55.59097  -2.648   0.0105 *
visit                    0.03397    0.09415 47.60058   0.361   0.7198  
age                     -0.09053    0.13425 37.86134  -0.674   0.5042  
sexMasculino            -0.18518    0.54147 43.04241  -0.342   0.7340  
hypertension1           -0.01098    0.27901 39.95460  -0.039   0.9688  
hypercholesterolemia1    0.13700    0.26340 35.97741   0.520   0.6062  
hypertrigliceridemia1    0.23126    0.29100 48.64692   0.795   0.4306  
drugs_w_loss1            0.33798    0.23290 35.20711   1.451   0.1556  
drugs_w_gain1           -0.43200    0.63860 54.38832  -0.676   0.5016  
mean_bp_mean             0.06963    0.11975 52.39903   0.581   0.5634  
handgrip                -0.18485    0.15881 46.02477  -1.164   0.2504  
evs_score                0.07079    0.10204 63.29260   0.694   0.4904  
alcohol_dose            -0.08050    0.08772 56.88145  -0.918   0.3626  
kcal                     0.07599    0.09918 60.05920   0.766   0.4466  
labs_alt                -0.18510    0.11358 59.17774  -1.630   0.1085  
labs_ggt                 0.25029    0.19634 51.20579   1.275   0.2081  
labs_ldl                 0.06425    0.12440 56.51527   0.516   0.6075  
labs_triglycerides      -0.09268    0.11221 63.29037  -0.826   0.4119  
labs_hdl                 0.09603    0.11454 53.33588   0.838   0.4056  
labs_quick_index        -5.82536    4.27972 60.08405  -1.361   0.1785  
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>
Correlation matrix not shown by default, as p = 22 &gt; 12.
Use print(x, correlation=TRUE)  or
    vcov(x)        if you need it</code></pre>
</div>
</div>
<p><em>2. Instrumental Variable (IV) Approach or Complier Average Causal Effect (CACE)</em></p>
<p>Use allocation_group as an instrument for treatment receipt/adherence, under the assumption that randomization affects inflammation only via the intervention. This estimates the effect among compliers, without the collider bias introduced by directly conditioning on post-randomization variables. This estimates the effect of actually completing the intervention, using random allocation as an instrument. In R, this can be implemented using <code>ivreg()</code> from the {AER} package or {ivtools}. The syntax is: outcome ~ endogenous + exogenous | instrument + exogenous. However, ivreg does not handle random effects. It assumes independent observations. Since our data has repeated measures (record_id), we should either use only one observation per subject (e.g., endpoint value) or use cluster-robust standard errors to account for within-subject correlation.</p>
<p>Collapsing longitudinal data to one observation per subject for instrumental variable (IV) analysis can be done in multiple ways, each answering a slightly different causal question. The gold standard in biostatistics is the ANCOVA approach, which models the outcome at the final time point while adjusting for the baseline value. This has been shown to be statistically more powerful and less biased than simple change scores when baseline and outcome are correlated. Like in the example:</p>
<pre><code>iv_model &lt;- ivreg(
  log1p(crp_final) ~ completed_intervention + log1p(crp_baseline) + covariates |
  allocation_group + log1p(crp_baseline) + covariates,
  data = data_final
)</code></pre>
<pre><code># Extract CRP at baseline
data_baseline &lt;- data_crp %&gt;%
  filter(visit == 1) %&gt;%
  select(record_id, labs_crp) %&gt;%
  rename(crp_baseline = labs_crp)

# Extract CRP at final visit
data_final &lt;- data_model_scaled %&gt;%
  group_by(record_id) %&gt;%
  filter(visit == max(visit)) %&gt;%  # This selects the final visit for each participant
  ungroup() %&gt;%
  select(
    record_id, labs_crp, crp_raw, completed_intervention, compliance_score_adjusted,
    duration_difference, allocation_group, age, sex, hypertension, hypercholesterolemia,
    hypertrigliceridemia, drugs_w_loss, drugs_w_gain, mean_bp_mean, handgrip, evs_score,
    alcohol_dose, kcal, labs_alt, labs_ggt, labs_ldl, labs_triglycerides, labs_hdl, labs_quick_index
  ) %&gt;%
  rename(crp_final = labs_crp)

data_cross_sectional &lt;- data_final %&gt;%
  left_join(data_baseline, by = "record_id")

rm(data_baseline)
rm(data_final)</code></pre>
<p><strong>Primary IV model that estimates the total effect</strong></p>
<p>The primary IV model that estimates the total effect is designed to evaluate the causal impact of the intervention on inflammation (CRP) as a whole, capturing both the direct effects and any indirect effects mediated by changes in other physiological or metabolic parameters. This model uses random assignment (allocation_group) as an instrument for actual participation or completion of the intervention (completed_intervention), allowing for the estimation of the Complier Average Causal Effect (CACE).</p>
<p>The intent is to answer whether the intervention, when adhered to, leads to a reduction in inflammation, regardless of the biological pathways involved. It reflects the overall therapeutic effect, incorporating all mechanisms through which the intervention may influence CRP. This approach does not adjust for variables that could be affected by the intervention itself (such as blood pressure, lipid levels, or liver enzymes), ensuring that the total causal effect is not inadvertently blocked or biased by post-treatment variables.</p>
</section>
<section id="iv" class="level4">
<h4 class="anchored" data-anchor-id="iv">IV</h4>
<pre><code>library(AER)

iv_model_total &lt;- ivreg(
  log1p(crp_final) ~ compliance_score_adjusted + log1p(crp_baseline) +
    age + sex + hypertension + hypercholesterolemia + hypertrigliceridemia |
  allocation_group + log1p(crp_baseline) +
    age + sex + hypertension + hypercholesterolemia + hypertrigliceridemia,
  data = data_cross_sectional
)

summary(iv_model_total)</code></pre>
<p><em>3. G-Computation / G-Formula</em></p>
<p>Simulate the outcome under a hypothetical scenario where everyone perfectly adheres to the intervention. This provides an estimate of the expected outcome if all participants had fully complied. R packages: {gfoRmula}, {ltmle}.</p>
<p><em>4. Marginal Structural Models (MSMs) with Inverse Probability Weighting (IPW)</em></p>
<p>Weight participants based on their probability of adhering, creating a pseudo-population where adherence is independent of confounders. This corrects for time-varying confounding affected by prior treatment. This gives an estimate of the intervention effect had everyone adhered as expected. Basic structure using {ipw}:</p>
</section>
<section id="ipw" class="level4">
<h4 class="anchored" data-anchor-id="ipw">IPW</h4>
<p>Complete case analysis (default approach in IPW):</p>
<pre><code>library(ipw)

data_model_ipw &lt;- data_model_scaled %&gt;%
  drop_na(
    completed_intervention,
    age, sex, hypertension, hypercholesterolemia, hypertrigliceridemia,
    drugs_w_loss, drugs_w_gain, mean_bp_mean, handgrip, evs_score,
    alcohol_dose, kcal, labs_alt, labs_ggt, labs_ldl, labs_triglycerides,
    labs_hdl, labs_quick_index
  )

model_weights &lt;- ipwpoint(
  exposure = completed_intervention,
  family = "binomial",
  link = "logit",
  denominator = ~ 
    age + sex + hypertension + hypercholesterolemia + hypertrigliceridemia +
    drugs_w_loss + drugs_w_gain + mean_bp_mean + handgrip + evs_score +
    alcohol_dose + kcal + labs_alt + labs_ggt + labs_ldl + labs_triglycerides +
    labs_hdl + labs_quick_index,
  data = data_model_ipw
)
</code></pre>
<p>Use simple imputation (if appropriate for wer data). Be cautious: imputation is a valid strategy if missingness is assumed to be Missing At Random (MAR) and the variables used for imputation are adequate.</p>
<pre><code>library(mice)
data_model_ipw &lt;- mice::complete(mice(data_model_scaled, m = 1))</code></pre>
<p><em>5. Bayesian Models with Compliance Modeling</em></p>
<p>Build a hierarchical Bayesian model that explicitly includes a latent compliance variable. Allows direct modeling of counterfactual scenarios under perfect adherence. Add prior distributions reflecting assumptions about compliance mechanisms. Packages: {brms}, {rstanarm}.</p>
<p>Example idea:</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb98"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb98-1"><a href="#cb98-1" aria-hidden="true" tabindex="-1"></a><span class="fu">brm</span>(<span class="fu">log1p</span>(labs_crp) <span class="sc">~</span> allocation_group <span class="sc">+</span> compliance_score_adjusted <span class="sc">+</span> visit <span class="sc">+</span> (<span class="dv">1</span> <span class="sc">|</span> record_id) <span class="sc">+</span> covariates,</span>
<span id="cb98-2"><a href="#cb98-2" aria-hidden="true" tabindex="-1"></a>    <span class="at">data =</span> data_model_scaled,</span>
<span id="cb98-3"><a href="#cb98-3" aria-hidden="true" tabindex="-1"></a>    <span class="at">family =</span> <span class="fu">gaussian</span>())</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p><em>Ways to Make we Current Model More Sensitive to the Intervention</em></p>
<ul>
<li>Focus the population: subset to high compliers or exclude partial compliers.</li>
<li>Model interactions: for example, allocation_group * compliance_score_adjusted to see whether the effect of the intervention depends on levels of compliance.</li>
<li>Use longitudinal modeling more fully: include random slopes for visit to capture differential trajectories over time.</li>
<li>Explore dose-response: instead of binary completed_intervention, use a continuous adherence measure (compliance_score_adjusted) to model the dose-response effect of adherence.</li>
<li>Consider transforming the outcome if distributional assumptions are strained (though log1p() is already a robust choice for skewed variables like CRP).</li>
</ul>
</section>
</section>
</section>
<section id="pam" class="level1">
<h1>PAM</h1>
<div class="cell">
<div class="sourceCode cell-code" id="cb99"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb99-1"><a href="#cb99-1" aria-hidden="true" tabindex="-1"></a>pam_1 <span class="ot">&lt;-</span> <span class="fu">lmer</span>(mean_bp_mean <span class="sc">~</span> (<span class="dv">1</span> <span class="sc">|</span> record_id) <span class="sc">+</span> visit <span class="sc">+</span> allocation_group <span class="sc">+</span> completed_intervention <span class="sc">+</span> duration_difference <span class="sc">+</span> age <span class="sc">+</span> sex <span class="sc">+</span> hypertension <span class="sc">+</span> hypercholesterolemia <span class="sc">+</span> hypertrigliceridemia <span class="sc">+</span> drugs_w_loss <span class="sc">+</span> drugs_w_gain <span class="sc">+</span> mean_bp_mean <span class="sc">+</span> handgrip <span class="sc">+</span> evs_score <span class="sc">+</span> alcohol_dose <span class="sc">+</span> kcal <span class="sc">+</span> labs_crp <span class="sc">+</span> labs_alt <span class="sc">+</span> labs_ggt <span class="sc">+</span> labs_ldl <span class="sc">+</span> labs_triglycerides <span class="sc">+</span> labs_hdl <span class="sc">+</span> labs_quick_index, <span class="at">data =</span> data_model_scaled)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning in model.matrix.default(fixedform, fr, contrasts): the response
appeared on the right-hand side and was dropped</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning in model.matrix.default(fixedform, fr, contrasts): problem with term 12
in model.matrix: no columns are assigned</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning in model.matrix.default(fixedform, fr, contrasts): the response
appeared on the right-hand side and was dropped</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning in model.matrix.default(fixedform, fr, contrasts): problem with term 12
in model.matrix: no columns are assigned</code></pre>
</div>
</div>
</section>

</main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    // For code content inside modals, clipBoardJS needs to be initialized with a container option
    // TODO: Check when it could be a function (https://github.com/zenorocha/clipboard.js/issues/860)
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->




</body></html>